Remnant: Face to Maw 1&2 (#4222)
endless-sky/endless-sky: data/remnant missions.txt

				decline
			
	on enter

> 		log "People" "Chilia" `As a military prefect, Chilia is responsible for the defence of Remnant space.`

Author: Agreed. And that information isn't really the right stuff for it to give, either. (The player learns that Chilia is a prefect, but he never says anything else about it)

Changed to: `Met Chilia, a Remnant Prefect, who explained why they wanted me to take a blood test.`

How does that sound? It is more appropriate to what the player has actually learned about him, and doesn't duplicate anything that gets added from the second log entry.

> 		log "People" "Chilia" `As an military prefect, Chilia is responsible for the defence of Remnant space.`

Reviewer: As a military

Author: Fixed. Thank!

> 		log "People" "Chilia" `Met Chilia, a Remnant Prefect, who explained why they wanted me to take a blood test.`

Author: Changed this sentence to not refer to "me".

> 		log "People" "Chilia" `Met Chilia, a Remnant Prefect, who explained why they want everyone to take a blood test.`

Reviewer 2: Logs for individual people aren't suppose to be written in the same way as normal logs. This sounds like a normal log to me, where as a `People` log for Chilia should be a description of who Chilia is, not something small that Chilia did.
If you're to rewrite this as a proper log *for* Chilia, add some information that isn't given elsewhere.
You could also just make this a normal log, but I'd still rewrite it to be something more like `Met Chilia, a Remnant Prefect, who explained that they blood test all foreign visitors because...`

Author: Thanks for the feedback. I've rewritten it. How does this look for a person log:
`Met Chilia, who is apparently a Remnant Prefect. He looks quite young to be holding a position of authority, but the Remnant I have seen seem to respect him.`

> 		log "People" "Chilia" `Met Chilia, who is apparently a Remnant Prefect. He looks quite young to be holding a position of authority, but the Remnant seem to respect him.`

Author: Ah. OK. Changed. (and thanks for providing an example.)

> 		log "People" "Chilia" `Met Chilia, who is apparently a Remnant Prefect. He looks quite young to be holding a position of authority, but the Remnant seem to respect him.`

Author: Fixed again.

		dialog "The research team quickly offloads the broken jump drive and pays you <payment> before hurrying back to their lab."

mission "Remnant: Face to Maw 1"
	name "Talk to Plume about the Void Sprites"
	description "Plume would be very interested in a report on what happened with the Void Sprites"
	landing
	to offer
		has "Remnant: Return the Samples 2: done"
	source
		system "Nenia"
	destination "Aventine"
	on offer
		conversation

> 			`As the swirls of gas close over the <ship> you wonder idly what brought you back here. This time, there is no apologetic researcher asking you to risk your life for research, or to return eggs from where they were stolen. The last glimpse of the Archon drifting overhead as you descend below the cloud deck provides no answers; For now, it is nothing more than a silent, brooding presence in the skies of Nenia. The silent depths of an alien world fold over you, gradually erasing the rest of the galaxy from sight.`

Reviewer 2: `For` -> `for`.

Author: Fixed. Good catch!

> 			`	As the <ship> levels off at the designated altitude, you strap yourself into the pressure suit. Stepping into the airlock, you lower the shields so you can exit onto the hull. Out on the surface, you admire the misty kaleidescope of colors filtering down through the cloud layers, playing over distant shapes moving in the mists. Before long several void sprites emerge, weaving and dancing among the clouds. They swirl for a few minutes around your cargo bay doors as if waiting for something then pulse briefly with flickers of light and then drop into the clouds below. The third void sprite flutters around the <ship>, poking at things and sliding over them. You quickly think to turn on your video recorder to catch the scene. The Void Sprite notices your brief movement, and turns towards you with a lunge. As it closes on you, your attention is drawn to what you suspect is its mouth: A zig-zagging crack that peels open like a zipper to reveal an opening that resembles an industrial grinder.`

Reviewer 3: `void sprite` -> `Void Sprite`

`into the` -> `into a` (too many instances of "the" in the sentence)

`if waiting for something then pulse` -> `if waiting for something, then pulse`

`and then drop` -> `and drop` or `and finally drop` (too many instances of "then" in the sentence)

`The third void sprite` -> `another void sprite` (it wasn't previously established that there were two; "several" is deliberately ambiguous)

Author: Thanks. Everything fixed.

> 			`	As the <ship> levels off at the designated altitude, you strap yourself into the pressure suit. Stepping into the airlock, you lower the shields so you can exit onto the hull. Out on the surface, you admire the misty kaleidescope of colors filtering down through the cloud layers, playing over distant shapes moving in the mists. Before long several void sprites emerge, weaving and dancing among the clouds. They swirl for a few minutes around your cargo bay doors as if waiting for something then pulse briefly with flickers of light and then drop into the clouds below. The third void sprite flutters around the <ship>, poking at things and sliding over them. You quickly think to turn on your video recorder to catch the scene. The Void Sprite notices your brief movement, and turns towards you with a lunge. As it closes on you, your attention is drawn to what you suspect is its mouth: A zig-zagging crack that peels open like a zipper to reveal an opening that resembles an industrial grinder.`

Reviewer 2: Do the shields need lowered? Shields are part of the hull itself, not something projected out into space.
Not all sci-fi shields prevent travel, either, so this is claiming something about shields in ES that I'm not sure is backed up right now.

Author: That's true. 

1) Shields in ES as described so far provide a purely military use, although some descriptions mention having been used historically for radiation and asteroid deflection. All these uses imply a physical barrier that prevents objects and energy from passing through it. 

2) It should also be pointed out that the gascraft is currently the only ship in the game that has gaslining, so this mission can only be triggered using it. And the gascraft is specifically described as a relic, so this description is only establishing lore regarding how antique remnant shields from centuries ago worked. These may well include the first barely-functional prototypes of alternate ship systems developed from alien ruins. so it is highly likely that they do not have all the functionality of modern shields.

3) I would also argue that a core feature of a gaslined ship is a hull constructed using different techniques. It is reasonable to think that such a hull may not incorporate shields in the same way that regular ships do. 

4) This is a research ship, and energy fields of any kind may interfere with precision equipment, especially those that try to measure minute energy fields. It makes sense to me that such a ship would include mechanisms to allow them to completely drop the shields for the sake of their study.

(caveat for 2, this mission leads to a second gaslining ship, but it is fundamentally impossible for the Penguin to be used in this mission without using an all-ships plugin. So the gascraft is the only gaslining ship the game that is possible to use for the mission)

> "Centuries ago, the Remnant built these ships in order to explore the upper atmosphere of nearby gas giants. The Archons took issue with the Remnant's research methods, and the studies were unable to be completed. The remaining gascraft have been gathering dust in storage for centuries while the Remnant waits to see if the anger of the Archons will subside."

Potential solutions:
- Add description making one or more of the above points explicit so that the player doesn't assume this lore is how all ES shields function. (this is my preferred solution, as I like the idea of having shields function differently for different groups. I see no reason why every race's shields should function identically)
- Re work the descriptions so that the player isn't disabling shields, but instead at the appropriate time is activating an emergency shield extension to protect themselves. (fits with the generic shield lore about variably shaped shields)
- The player activates a plasma cutting torch on their suit and chops off the tentacle. (Possible, but the game makes a point of the character not being particularly adept at physical combat related things)

Author: While it doesn't fully resolve this debate, some of the conversation in the Tech Retrieval mission rewrite (#4269 ) includes some commentary on the gascraft, and mentions how said ship uses a lot of archaic technology, including 'hard-shell' shields.

> "You've flown the gascraft, you know how old that ship is: primitive energy systems, antique 'hard-shell' shields that have to be manually raised and lowered..." she trails off into momentary silence. "Be glad that you got one that had already been refurbished prior to your visit to Nenia. At least it had modern engines and an updated sensor array."

> 			`	Up close, the void sprite is massive: Most of its tentacles are easily as thick as your leg, and move with surprising speed and precision. It moves through the air like a strange kind of fish and you can make out a low vibration you suspect is probably from the air whistling over it as it closes with you. Facing down this rapidly approaching giant, you feel something click in the back of your mind and a wave of paralysis washes over you. In a moment of lucidity moments before the void sprite hits, you recall that your shields are keyed to your equipment: You could activate them to protect yourself from this attack, and they would extend around you. It probably wouldn't be too healthy for the Void Sprite, though.`

Reviewer 3: `void sprite` -> `Void Sprite`

`It moves through the air` Too many instances of the verb "to move" in close proximity

Author: Thanks! Changed one of the "moves" into "shifts".

			choice
				`	(Activate shields)`

> 				`	(Do not activate shields)`

Reviewer 2: Missing periods in these choices.

Author: Fixed! Thanks!

					goto holdground

> 			`	Your mind latches onto the idea and you trigger the shields in a rush. The millisecond that follows feels like an eternity: The void sprite is just a few meters away, one tendril reaches out and touches you just as the shield slams into place. You see it strike the tendril and severs it cleanly in the process of re-establishing a sealed barrier. The same instant that the pressure on your mind with the paralysis turns off, as if it was simply a dream. You momentarily stare at the immobile void sprite, both frozen in shock over what just happened. Then time reasserts itself and sprite spasms away from the ship with an unearthly crescendo of tones. It plunges down into the clouds, leaving you standing on your hull with the floating severed piece of tendril.`

Reviewer 4: Just going to say that shields are meant to be very close to your ship, I wouldn't have though that you could: 
A: extend it in any way or
B: have it cut off about 1m of anything.

Author: The lore doesn't seem too clear on that - and ships have a lot of greebles to warp around. I don't see too much difference between a turret that moves around and a heavy duty pressure suit that is magnetically clamped to the hull. (The robots from a System's core aren't affected, either.)

Based on a rather loose interpretation of the generators vs re-generators descriptions, I've generally taken the view that primitive shields and those produced by generators tended to be spherical (not perfect spheres, probably egg shaped in most cases) as that is the easiest, whereas re-generators and more advanced shielding tended to hug the ship more, which costs more energy but is more flexible.

But that is something that should be discussed. If something is projecting through where the shield will be, what happens to it? Could a piercer be used to weaken a shield, something be inserted through, and then used thereafter as a conduit for getting stuff into or out of a shielded area? Would it simply be sheared off? Or would it be eroded over time, kind of like a laser burning its way through a blast door?

Reviewer 3: `same instant that the` -> `same instant, the`

Author: Thanks, fixed!

			`	As you re-enter the ship, you notice that your suit is covered in some kind of translucent slime. You scrape off as much as you can into a sealed container, but your pressure suit appears to be permanently stained an iridescent purplish color as a result. You resolve to go find Plume, since he will almost certainly find this interesting.`
				accept
	on accept
		fail "Remnant: Face to Maw 1b"
	on complete
		conversation
			`When you walk into Plume's lab on Aventine you find him buried in some kind of holographic projection of your egg returning mission. You can see that he appears to be watching the video in slow-motion as he taps out notes on his datapad. After a few minutes he looks up and see you standing in the entryway. He quickly pulls himself out of the projection and hits what you assume must be the pause button.`
			`	"Captain <first>! What a pleasant surprise," he trills. "If you are looking for more work, I don't have anything for you right now."`

> 			`	"Actually, I have something for you," you reply. "I visited Nenia and had an interesting encounter with a void sprite. I came away from it with some more footage and a container of slime." He eagerly accepts the datacube and the container from you. He slides the container into some kind of machine that immediately starts whirring, and slots the datacube into his console. Moments later his holographic projection shifts to the new footage. He checks the time stamps, then chants "These will take me a little while to analyze. I will meet you in the spaceport in a few hours to discuss my preliminary findings."`

Reviewer 2: This feels a little off to me. I think that in situations like this, typically the conversation refers to you speaking without direct dialog. E.g. in this scenario, it may instead be `You tell Plume about how you visited Nenia and had an interesting encounter with a void sprite...`

Author: Rewrote this section to remove the direct dialogue on the part of the player.

mission "Remnant: Face to Maw 1b"
	name "Talk to Plume about the Void Sprites"
	description "Plume would be very interested in a report on what happened with the Void Sprites"
	landing
	to offer
		has "Remnant: Return the Samples 2: done"
	to fail
		or
			has "Remnant: Face to Maw 1: done"
			has "Remnant: Face to Maw 1: active"
	to complete
		has "Remnant: Face to Maw 1: declined"
	source
		system "Nenia"
	destination "Aventine"
	on complete
		conversation
			`When you walk into Plume's lab on Aventine you find him buried in some kind of holographic projection of your egg returning mission. You can see that he appears to be watching the video in slow-motion as he taps out notes on his datapad. After a few minutes he looks up and see you standing in the entryway. He quickly pulls himself out of the projection and hits what you assume must be the pause button.`
			`	"Captain <first>! What a pleasant surprise," he trills. "If you are looking for more work, I don't have anything for you right now."`

> 			`	"Actually, I have something for you," you reply. "I visited Nenia and a void sprite attacked me. I came away from it with some more footage for you and a this tentacle" you gesture at the container. Plume recoils from you in disgust. "After all our work rebuilding trust with the Archon and the Void Sprites, you go and risk it for... what, exactly? Sightseeing? Thrill of the hunt?" his chant settles into an angry tone. "Leave the recording on the console. I want no part of that tentacle, though."`

Reviewer 3: `and a this tentacle` -> `and this tentacle,`

Author: Fixed.

			`	As you are leaving Plume's lab a technician pulls you aside. "Our records mention that the tentacles had some interesting properties that were useful in combat. If you are interested, meet me in the spaceport in a few hours."`

mission "Remnant: Face to Maw 2"

> 	name "Take the slime sample to Tali on Viminal"

Reviewer 4: If the player hasn't met Tali then this is an odd mission name.

Author: That's true. Changed.

	description "The slime has properties of interest to a team of engineers on <planet>."
	to offer
		has "Remnant: Face to Maw 1: done"
	source "Aventine"
	destination "Viminal"
	cargo "slime sample" 1
	on offer
		conversation

> 			`When Plume finally arrives you can tell from his step that he is excited. He slides onto the bench opposite you and pulls out a datapad. "I admire your bravery, Captain. I don't think I would have the courage to hold my ground against a void sprite like that" he chants respectfully, then changes tone to one of wonder. "In that last moment it almost looked like it was sizing you up or examining you." He shakes his head with an amused expression. "I spend so much time studying non-human creatures I sometimes slip and start anthropomorphizing them." His voice takes on a more militant cadence. "That was probably some kind of territorial display, or possibly some kind of dominance challenge. We don't know enough about void sprite behaviors to speculate further, though."`

Reviewer 3: `like that` -> `like that,`

Author: Fixed.

			`	"Be that as it may be, the more immediate reward for your curiosity is in this sample," he trills with excitement. "Several of our labs on Viminal have been working on the development of a new generation of ship technology, and this substance contains some chemicals that could be a key component. I have already transmitted the data to them, but they would like to compare with the original. Could you take this sample to them?"`
			choice
				`	"Yes, I can."`
					accept
				`	"I would rather not."`
			`	Plume stoically accepts that with a nod "That is your choice. I will arrange for another ship to transport it."`
			decline
	on accept
		event "remnant: penguin"
	on complete

> 		log "People" "Tali" `As an engineering prefect, Tali is the foremost authority among the Remnant on salvaged technology and a senior leader in their reverse engineering program. She is also a prominent engineer in the field of spacecraft propulsion.`

Author: Completely rewrote the log entry:
`One of Tali's main priorities is improving the maneuverability characteristics of Remnant Ships. To this end she has been working on building a new prototype ship based on the void sprites.`

That should stack nicely on the previous introductory log entries.

> 		"remnant tali" ++

Author: I could do that, yes. Given that it needs the second Void Sprite mission chain to be complete, and then have the player go back to "just visit" Nenia anyway, I think it is rather likely that they have already met her anyway, but I like giving people the chance to do things in different orders, when possible.

This isn't a major issue, though.  If the branch there ends up causing problems or can't be resolved to work/ read right, then I'll scrap all that and just put in the "already met Tali" requirement.

		payment 1000000
		conversation
			branch tali
				"remnant tali" > 1

> 			`When you land on Viminal a solidly-built female Remnant in coveralls is waiting for you. "Greetings, Captain <first>. My name is Tali, and I am a Prefect among the Remnant. Above all else, I study motion, and in particular how our starships move."`

Reviewer 3: ![31b-me2-021717](https://user-images.githubusercontent.com/1097249/55294519-55cf8380-53c0-11e9-823c-9207f721e1d4.jpg)

Female engineer named Tali who's fascinated with starships == an existing Mass Effect character. She should probably have a different name. :)

Reviewer 2: >She should probably have a different name. :)

We can have references? :)
There was a conversation about it in #3761.

Starts here: https://github.com/endless-sky/endless-sky/pull/3761#discussion_r243917477

Reviewer 3: Yeah but this isn't a reference, it's a straight-up rip-off. :) I mean you wouldn't create a character named Darth Vader who dresses in all black, wears a mask, and leads the enemy forces, would you?

Reviewer 2: Idk, does this Darth Vader also carry around a laser sword, can use magic to effect things at a distance, and is the father of the main character? I've never played Mass Effect, but I'm sure there's more to Tali's character aside from being female and into spaceships.

Author: Name, female, engineer, and focused on starships is about the extent of the similarities.

In terms of personality, Mass Effect Tali starts out quite young, shy, usually reserved, and is someone just starting out in the world, trying to make a name for herself. Later in the series she develops some self-confidence and becomes a leader. She often flirts with the protagonist (and is one of the characters the player can become romantically involved with). In-game she is what I'd call a "tweaker engineer" - as in, she spends most of her time adjusting, monitoring, and fine-tuning things. Which fits with the Quarian culture, as they are a ship-bound nation that doesn't have any planets or easy access to any kind of resources. So it is rare that they build anything new, and they spend most of their time maintaining and tweaking their current ships - most of which are many centuries old.

Remnant Tali starts out as a confident, mature person who is already well established in terms of her position and abilities. She has her own interests and life, which do not revolve around the protagonist, and is not flirtatious in the least. In-game, she is portrayed as being  a hands-on inventor: Building ships, installing new things, designing new ships, etc. The Remnant have access to three planet's worth of resources, including ones with functioning advanced alien technology, rare crystals, and other advantages. They are described as being fast innovators who managed to develop and build new warships based on the alien tech they found in very short order, so being  a designer and builder fits better.

Physically speaking, Mass Effect Tali has a slight, curvaceous silhouette, and the player never sees anything underneath that environment suit. ME Tali is effectively immuno-compromised (hence the suit), is physically weak, and not particularly hardy. In-game, Tali is most often found leaning over control consoles, pondering the read-outs. She is also quite young, and is described as being in the first few years of adult-hood. When the player initially meets her, she is in the midst of her pilgrimage - a rite of passage to demonstrate that she is an adult and prove her worth to society. Even by the end of the series (which at most is 3-4 years later) she is still one of the youngest Quarians that the player interacts with.

Remnant Tali is described physically as being a sturdy and solidly-built individual who is comfortable in short sleeves even in the near freezing temperatures of Viminal. In-game, Tali is usually found in some sort of physically active scene - climbing over ships, installing components, tightening things with wrenches, etc. She is an adult approaching mid-life: Still very physically fit, but isn't brash about it the way younger people often are.

---

In short, the Tali in here is a deliberate tribute to the Mass Effect Tali, but one that is intentionally designed to chart her own path with her own identity in terms of both her personality and physicality. For players that see the name and think Tali from ME, I intend her to be memorable in how she will clash with their expectations, and that will include descriptions of her doing things that the ME Tali never did. Basically, taking the player's expectation of finding the slim vulnerable Tali from ME, and instead finding what amounts to Endless Sky's equivalent of Rosie the Riveter.

![steampunk-rosie-the-riveter-we-can-do-it](https://user-images.githubusercontent.com/32169904/55670443-8771a000-5841-11e9-9367-98f3fa3df6f5.png)
(Not the original one, I know, but it was the closest thing I could find searching for "Sci Fi Rosie the Riveter")

Reviewer 3: I get what you're doing, I just don't think it works. Whenever I read the name I'm unable to avoid my brain jumping to ME Tali.

Reviewer 4: Your Tali will always be the ME Tali to me too, regardless of how much you try to push the differences.

Author: I get that.

Should I change the spelling, then? Talie, perhaps?

Author: OK, all instances of "Tali" have been replaced with "Taly". Does that work?

Author: @Pointedstick @jafdy One thing to point out, however. Tali's already in the story as Tali for a few months now. How many saves will that potentially confuse? It won't break any saves or progression, but it might trigger an extra "haven't met Taly before" branch and an extra log entry.

Reviewer 4: Non-live version, I think it'll be fine. (And I would like to say that I objected to this in the first instance.)

Author: Yes, and if I recall correctly, ended up getting battered into submission by a variety of people saying it'll be fine. But if after a few months of ES having a tali too, you are still associating Tali with a frail immune-compromised alien instead of a strong dark skinned engineer, well, time to change it before too many people get used to it.

Author: #4254 Should solve this for you.

			goto main
			label tali
			`	When you land on Viminal you spot Tali waiting for you. "Greetings, Captain <first>. It is good to see you again."`
			label main

> 			`	"So, Plume sent us some very interesting data, along with a message to expect you with a sample?" You hand her the container with the slime sample and she holds it up to the light. "Interesting looking stuff" she chants. "Let's go back to my shop." She leads the way through several side passages to an out-of-the-way landing pad connected to a large hanger. Inside, you can see a giant tank filled with a murky fluid, watched over by a flock of displays. She makes her way over to a large machine with a several slots which she slides the container into. As the machine starts to work, she turns back to you.`

Reviewer 3: `stuff` -> `stuff,`

`watched over by a flock of displays` Do displays really watch things? Also, not sure "flock" is the correct term. How about something like `surrounded by an enormous bank of displays` instead?

Author: Fixed. Changed to `, surrounded by equipment and displays.`

> 			`	"So, Plume sent us some very interesting data, along with a message to expect you with a sample?" You hand her the container with the slime sample and she holds it up to the light. "Interesting looking stuff" she chants. "Let's go back to my shop." She leads the way through several side passages to an out-of-the-way landing pad connected to a large hanger. Inside, you can see a giant tank filled with a murky fluid, watched over by a flock of displays. She makes her way over to a large machine with a several slots which she slides the container into. As the machine starts to work, she turns back to you.`

Reviewer 3: `a several slots` -> `several slots`

Author: fixed.

> 			`	"As the primary example of space-faring life, the Void Sprites have always been of interest to me. They have numerous advantages that we as atmosphere-bound creatures can only dream of, such as being able to experience space in a much more direct manner. I have been working on a side project to build a new generation of ships that melds these ideas with our current technology, but the data we have on the Void Sprites is centuries old and incomplete" her chant trails off as she pulls up a display with your original scans on them. "However, your recent explorations provided new data I could use to update my designs. This slime that you retrieved is the final ingredient."`

Reviewer 3: `incomplete` -> `incomplete,`

Author: Thanks, fixed.

> 			`	The machine dings, and she looks over the results and then trills excitedly. "Yes, it matches! So it is time to begin the final activation." She strides over to a central console and taps in a series of commands, followed by an iris scan and a blood prick. The industrial hum of the room suddenly ramps up to a crescendo as your senses are assaulted by what you can only describe as the smell of a thunderstorm in a swamp. A few seconds later it begins to quieten again, and the tank starts lowering into the floor as a shape emerges from the liquid.`

Reviewer 3: `senses are assaulted` Technically only your noise is being assaulted by the smell. :)

Author: That's true. I was originally thinking about the noise also affecting one's ears.

> 			`	The machine dings, and she looks over the results and then trills excitedly. "Yes, it matches! So it is time to begin the final activation." She strides over to a central console and taps in a series of commands, followed by an iris scan and a blood prick. The industrial hum of the room suddenly ramps up to a crescendo as your senses are assaulted by what you can only describe as the smell of a thunderstorm in a swamp. A few seconds later it begins to quieten again, and the tank starts lowering into the floor as a shape emerges from the liquid.`

Reviewer 3: `quieten` I would recommend using a more common word here.

Author: How about `A few seconds later the noise softens,`

			`	The ship before you has some faint traces of the gascraft you have flown to Nenia, along with other elements of Remnant design, but it is clearly something new. Its antennae swivel around, as if taking in its surroundings, and the tendrils along the side twitch and reach out to touch its surroundings. Solidly built tendrils underneath it flex as it lifts itself off its supports and begins to move around. You have a momentary vision of what this ship might look like in the skies of Nenia.`
			scene "scene/penguinscene"

> 			`	Tali's droning chant of technical details finally seeps through your revery: "... so from a captain's point of view flying the penguin should be very similar to a normal ship, except she flies herself doesn't need a crew to make her work." She pauses, and looks at you. "Your curiosity provided the final ingredients to bring her to life. You should be the first to have one. You'll still have to go through the shipyard to register her and pay the resource cost to balance the logistics, but this first one is reserved for you." She pats the nose of the penguin and it makes a wooshing noise, then starts going over the ship with a scanner, leaving you to contemplate in silence.`

Reviewer: except it flies itself and doesn't require a crew to make itself work."

> She pats the nose of the penguin and it makes a wooshing noise, then starts going over the ship with a scanner, leaving you to contemplate in silence.`
 This just seems extremely off to me, edit it.

Author: The whole paragraph, or just the last sentence?

I changed the last sentence to:
`She pats the nose of the penguin and it makes a wooshing noise. After a moment she starts going over the ship with a scanner, leaving you to contemplate in silence.`

I think it still needs a bit of refining, but it feels better than the original.

> 			`	Tali's droning chant of technical details finally seeps through your revery: "... so from a captain's point of view flying the penguin should be very similar to a normal ship, except she flies herself doesn't need a crew to make her work." She pauses, and looks at you. "Your curiosity provided the final ingredients to bring her to life. You should be the first to have one. You'll still have to go through the shipyard to register her and pay the resource cost to balance the logistics, but this first one is reserved for you." She pats the nose of the penguin and it makes a wooshing noise. After a moment she starts going over the ship with a scanner, leaving you to contemplate in silence.`

Reviewer 3: `revery` -> `reverie`

`flies herself doesn't need` -> `flies herself and doesn't need`

Author: Thanks! changed.

event "remnant: penguin"
	shipyard "Remnant Gascraft"
		add "Penguin"

mission "Remnant: Face to Maw 2b"
	name "Take the tentacle to Chilia's engineers on Caelian"
	description "The tentacle has properties that may be put to use by the engineers on <planet>."
	to offer
		has "Remnant: Face to Maw 1b: done"
	source "Aventine"
	destination "Caelian"
	cargo "tentacle" 1
	on offer
		conversation
			`When the technician arrives in the spaceport he immediately strides over to you. "I checked the records, and my memory was correct. One of our research teams back then discovered that these tentacles could be used as a devastating weapon in close quarters combat."`

> 			`	He pulls out an image of a strange looking rifle and continues to chant "the personal weapons lab under Prefect Chilia has a few of the old weapon frames. If you go there, they could turn this tentacle into a fairly potent weapon for you.`

Reviewer 4: No closing `"`

Author: Fixed.

> 			`	He pulls out an image of a strange looking rifle and continues to chant "the personal weapons lab under Prefect Chilia has a few of the old weapon frames. If you go there, they could turn this tentacle into a fairly potent weapon for you."`

Reviewer 3: `chant` -> `chant,`

Author: fixed.

			choice
				`	"That sounds good."`
					accept
				`	"I would rather not."`

> 			`	Plume stoically accepts that with a nod "That is your choice. Good hunting."`

Reviewer 4: I thought Plume wasn't dealing with the tentacle.

Author: You're right, he isn't. Thanks! fixed.

			decline
	on complete

> 		log "People" "Chilia" `As an military prefect, Chilia is responsible for the defence of Remnant space. He can usually be found on the front lines of combat unless required to work on tactics for the Remnant as a whole.`

Author: Fixed by rewriting and changing the previous log entry so that it no longer provides anything included in this entry.

		payment 250000
		"remnant chilia" ++
		outfit "Void Rifle"
		conversation
			branch chilia
				"remnant chilia" > 1

> 			`You follow the directions provided you and eventually find yourself at a Remnant weapons lab. You enter and are met by security, who ask you to wait. A few minutes later a fit young man emerges from another door "Greetings, captain <first>. I am prefect Chilia. I have the nominal responsibility for the overall defense of Remnant space and military operations." He smiles depreciatingly "I say 'nominal' as virtually all our activities are military in some way, but usually fall under the purview of those who better understand them."`

Reviewer 3: `from another door` This sentence needs to end with punctuation, and needs either another clause or another sentence to indicate that Chilia is starting to speak. Like this:
```
A few minutes later a fit young man emerges from another door. "Greetings, captain <first>," he sings. "I am prefect Chilia. I have the nominal responsibility..."
```

`He smiles depreciatingly` -> `He smiles depreciatingly.`

Author: Thanks, fixed.

				goto main
			label chilia

> 			`You follow the directions provided you and eventually find yourself at a Remnant weapons lab. You enter and are met by security, who ask you to wait. A few minutes later a fit young man emerges from another door "Greetings, captain <first>, we meet once again." You immediately recognize the voice as being the man who convinced you to provide the blood sample and get to know the Remnant. "We didn't get a proper introduction last time, but you have probably figured out that I have the nominal responsibility for the overall defense of Remnant space and military operations." He smiles depreciatingly "I say 'nominal' as virtually all our activities are military in some way, but usually fall under the purview of those who better understand them."`

Reviewer 3: Ditto

Author: Fixed.

			label main

> 			`	He leads you to a room that looks like an armory where a team of techs are putting the finishing touches on a rifle stock. The request from the tech on your behalf caught my attention, as we haven't had the key resources for making void rifles in centuries. Fortunately, we haven't really needed them since our starships have been brought up to their current capabilities." He gestures, and a tech takes the container with the tentacle from you and starts installing it in a framework.`

Reviewer 4: No leading `"`

Author: Good catch. Thanks!

			choice
				`	"Void rifle?"`

> 				`	(expression of boredom)`

Reviewer 2: This doesn't follow the usual action choice format. (It should be an actual sentence.)

Author: I changed it to "I do not need details."

That being said, would (Make an expression of boredom.) sound better?

					goto end

> 			`	"Yes, void rifle. I don't know who discovered it, but someone figured out that if a particularly high charge was poured through the tentacles nervous system in a specific pattern, it emits a paralyzing cone of energy with a reasonably decent range. I'm not an engineer, so I do not really understand the mechanics of it." He pauses to watch as a tech installs the now-caged tentacle on top of the stock. "Suffice to say, it is useless for sniping, but its ability to shoot through walls and affect areas makes it excellent for clearing ships and tight quarters. Back when we didn't have the ships to resist the Korath, most of our battles were resolved in building to building combat. These rifles gave us the advantage we needed to hold our own until we could even the battlefield up there." He gestures upwards.`

Reviewer 3: `tentacles` -> `tentacle's`

Author: Fixed.

			label end

> 			`	"Enough of history, though." He turns to the techs who are finishing the assembly. "The tentacle has been treated to resist decay and the rifle itself will provide handle basic maintenance routines for as long as you charge it regularly." He strides over, picks up the completed void rifle, and then hands it to you. "You are holding a piece of our history, Captain. Use it well!"`

Reviewer 4: `...will provide handle...` Do you mean one of those or a `[provide/handle]`

Author: just one or the other. Fixed.

> 			`	"Enough of history, though." He turns to the techs who are finishing the assembly. "The tentacle has been treated to resist decay and the rifle itself will provide basic maintenance routines for as long as you charge it regularly." He strides over, picks up the completed void rifle, and then hands it to you. "You are holding a piece of our history, Captain. Use it well!"`

Reviewer 3: `for as long as you charge it regularly` would seem to imply that the weapon should actually consume some ship energy, but it doesn't (I think that would be really cool).

Author: I could definitely add that, although I'm not sure what value would be reasonable.

On the flip side, I"m not sure a unique h2h weapon should be drawing enough power to be significant or noticeable on any ship other than those with the smallest or weakest of generators. Ideally it'd probably be more of a "lump sum" draw after the player has engaged in h2h combat, but we don't have mechanics for that, as far as I can tell.

Author: OK, for now I have given it `"Energy Consumption" 0.01` That's quite low - probably insignificant, but it is there.

	landing
	to offer
		has "Remnant: Return the Samples 2: done"
	to fail
		has "Remnant: Face to Maw 1: done"
		has "Remnant: Face to Maw 1: active"
	to complete
		has "Remnant: Face to Maw 1: declined"
	source
		system "Nenia"
	destination "Aventine"
	on complete
		conversation

> 			`When you walk into Plume's lab on Aventine you find him berried in some kind of holographic projection of your egg returning mission. You can see that he appears to be watching the video in slow-motion as he taps out notes on his datapad. After a few minutes he looks up and see you standing in the entryway. He quickly pulls himself out of the projection and hits what you assume must be the pause button.`

Reviewer 5: "berried" should be "buried".

Author: Should already be fixed, but I'll check again. Thanks!

		dialog "The research team quickly offloads the broken jump drive and pays you <payment> before hurrying back to their lab."

mission "Remnant: Face to Maw 1"

> 	name "Talk to Plume about the Void Sprites"

Reviewer 3: Un-capitalize Void Sprites

Author: Fixed. Thanks!

	description "Plume would be very interested in a report on what happened with the void sprites"
	landing
	to offer
		has "Remnant: Return the Samples 2: done"
	source
		system "Nenia"
	destination "Aventine"
	on offer
		conversation
			`As the swirls of gas close over the <ship> you wonder idly what brought you back here. This time, there is no apologetic researcher asking you to risk your life for research, or to return eggs from where they were stolen. The last glimpse of the Archon drifting overhead as you descend below the cloud deck provides no answers; for now, it is nothing more than a silent, brooding presence in the skies of Nenia. The silent depths of an alien world fold over you, gradually erasing the rest of the galaxy from sight.`

> 			`	As the <ship> levels off at the designated altitude, you strap yourself into a pressure suit. Stepping into the airlock, you grimace at the antiquity of these hard-shell shield systems, and tap the 'lower shields' button so you can exit onto the hull. Out on the surface, you admire the misty kaleidescope of colors filtering down through the cloud layers, playing over distant shapes moving in the mists. Before long several void sprites emerge, weaving and dancing among the clouds. They swirl for a few minutes around your cargo bay doors as if waiting for something, then pulse briefly with flickers of light and drop into the clouds below. Another void sprite flutters around the <ship>, poking at things and sliding over them. You quickly think to turn on your video recorder to catch the scene. The void sprite notices your brief movement, and turns towards you with a lunge. As it closes on you, your attention is drawn to what you suspect is its mouth: A zig-zagging crack that peels open like a zipper to reveal an opening that resembles an industrial grinder.`

Reviewer 3: `that resembles an` -> `resembling`

Author: Changed!

	on complete
		conversation
			`When you walk into Plume's lab on Aventine you find him buried in some kind of holographic projection of your egg returning mission. You can see that he appears to be watching the video in slow-motion as he taps out notes on his datapad. After a few minutes he looks up and see you standing in the entryway. He quickly pulls himself out of the projection and hits what you assume must be the pause button.`
			`	"Captain <first>! What a pleasant surprise," he trills. "If you are looking for more work, I don't have anything for you right now."`
			`	You tell Plume about how you visited Nenia and had an interesting encounter with a void sprite, and offer him the recording and container of slime. He eagerly accepts the datacube and the container from you. He slides the container into some kind of machine that immediately starts whirring, and slots the datacube into his console. Moments later his holographic projection shifts to the new footage. He checks the time stamps, then chants "These will take me a little while to analyze. I will meet you in the spaceport in a few hours to discuss my preliminary findings."`

mission "Remnant: Face to Maw 1B"

> 	name "Talk to Plume about the Void Sprites"

Reviewer 3: Un-capitalize Void Sprites

Author: Fixed.

			`	"Be that as it may be, the more immediate reward for your curiosity is in this sample," he trills with excitement. "Several of our labs on Viminal have been working on the development of a new generation of ship technology, and this substance contains some chemicals that could be a key component. I have already transmitted the data to them, but they would like to compare with the original. Could you take this sample to them?"`
			choice
				`	"Yes, I can."`
					accept
				`	"I would rather not."`
			`	Plume stoically accepts that with a nod "That is your choice. I will arrange for another ship to transport it."`
			decline
	on accept
		event "remnant: penguin"
	on complete

> 		log "People" "Taly" `One of Taly's main priorities is improving the maneuverability characteristics of Remnant Ships. To this end she has been working on building a new prototype ship based on the void sprites.`

Reviewer 3: Needs to use the new spelling

Author: Fixed. Thanks!

			`	The machine dings, and she looks over the results and then trills excitedly. "Yes, it matches! So it is time to begin the final activation." She strides over to a central console and taps in a series of commands, followed by an iris scan and a blood prick. The industrial hum of the room suddenly ramps up to a crescendo as your nose is assaulted by what you can only describe as the smell of a thunderstorm in a swamp. A few seconds later the noise softens, and the tank starts lowering into the floor as a shape emerges from the liquid.`

> 			`	The ship before you has some faint traces of the gascraft you have flown to Nenia, along with other elements of Remnant design, but it is clearly something new. Its antennae swivel around, as if taking in its surroundings, and the tendrils along the side twitch and reach out to touch its surroundings. Solidly built tendrils underneath it flex as it lifts itself off its supports and begins to move around. You have a momentary vision of what this ship might look like in the skies of Nenia.`

Reviewer 3: `tendrils` used twice in close proximity

Author: Changed the first tendrils to be tentacles.

			label main
			`	He leads you to a room that looks like an armory where a team of techs are putting the finishing touches on a rifle stock. "The request from the tech on your behalf caught my attention, as we haven't had the key resources for making void rifles in centuries. Fortunately, we haven't really needed them since we resolved the threats that originally plagued our settlements.`
			choice
				`	"It is called a Void Rifle?"`

> 				`	"I do not need details."`

Reviewer 3: Both of these dialog options are worded to sound very formal to me. Maybe use contractions and re-word a bit?

Author: Switched in the contractions. Is that enough, or should I reword too?

Reviewer 3: Yep, looks good enough!

Author: Thanks!

					goto end

> 			`	"Yes, Void Rifle. I don't know who discovered it, but someone figured out that if a particularly high charge was poured through the tentacle's nervous system in a specific pattern, it emits a paralyzing cone of energy with a reasonably decent range. I'm not an engineer, so I do not really understand the mechanics of it." He pauses to watch as a tech installs the now-caged tentacle on top of the stock. "Suffice to say, it is useless for sniping, but its ability to shoot through walls and affect areas makes it excellent for clearing ships and tight quarters. Back then we had a number of threats that made building-to-building clearing operations an unfortunately frequent occurance." His grim tone catches your attention, but he seems unwilling to pursue the topic.`

Reviewer 3: `Suffice to say` -> `Suffice it to say`

Author: Fixed. Thanks!

			label end

> 			`	"Enough of history, though." He turns to the techs who are finishing the assembly. "The tentacle has been treated to resist decay and the stock mechanisms will provide basic maintenance routines for as long as you charge it regularly." He strides over, picks up the completed void rifle, and then hands it to you. "You are holding a piece of our history, Captain. Use it well!"`

Reviewer 3: `Enough of history, though` -> `All right, enough history`

Author: Changed.

		dialog "The research team quickly offloads the broken jump drive and pays you <payment> before hurrying back to their lab."

mission "Remnant: Face to Maw 1"
	name "Talk to Plume about the void sprites"

> 	description "Plume would be very interested in a report on what happened with the void sprites"

Author: Thanks. Fixed.

	landing
	to offer
		has "Remnant: Return the Samples 2: done"
	source
		system "Nenia"
	destination "Aventine"
	on offer
		conversation
			`As the swirls of gas close over the <ship> you wonder idly what brought you back here. This time, there is no apologetic researcher asking you to risk your life for research, or to return eggs from where they were stolen. The last glimpse of the Archon drifting overhead as you descend below the cloud deck provides no answers; for now, it is nothing more than a silent, brooding presence in the skies of Nenia. The silent depths of an alien world fold over you, gradually erasing the rest of the galaxy from sight.`

> 			`	As the <ship> levels off at the designated altitude, you strap yourself into a pressure suit. Stepping into the airlock, you grimace at the antiquity of these hard-shell shield systems, and tap the 'lower shields' button so you can exit onto the hull. Out on the surface, you admire the misty kaleidescope of colors filtering down through the cloud layers, playing over distant shapes moving in the mists. Before long several void sprites emerge, weaving and dancing among the clouds. They swirl for a few minutes around your cargo bay doors as if waiting for something, then pulse briefly with flickers of light and drop into the clouds below. Another void sprite flutters around the <ship>, poking at things and sliding over them. You quickly think to turn on your video recorder to catch the scene. The void sprite notices your brief movement, and turns towards you with a lunge. As it closes on you, your attention is drawn to what you suspect is its mouth: A zig-zagging crack that peels open like a zipper to reveal an opening resembling an industrial grinder.`

Author: Fixed, and broken up into two paragraphs.

I'll keep that in mind for the future. Thanks!

> 			`	Up close, the void sprite is massive: Most of its tentacles are easily as thick as your leg, and move with surprising speed and precision. It shifts through the air like a strange kind of fish and you can make out a low vibration you suspect is probably from the air whistling over it as it closes with you. Facing down this rapidly approaching giant, you feel something click in the back of your mind and a wave of paralysis washes over you. In a moment of lucidity moments before the void sprite hits, you recall that your shields are keyed to your equipment: You could activate them to protect yourself from this attack, and they would extend around you. It probably wouldn't be too healthy for the void sprite, though.`

Author: Fixed. Thanks!

			choice
				`	(Activate shields.)`
				`	(Do not activate shields.)`
					goto holdground

> 			`	Your mind latches onto the idea and you trigger the shields in a rush. The millisecond that follows feels like an eternity: The void sprite is just a few meters away, one tendril reaches out and touches you just as the shield slams into place. You see it strike the tendril and severs it cleanly in the process of re-establishing a sealed barrier. In the same instant, the pressure on your mind vanishes, as if it was simply a dream. You momentarily stare at the immobile void sprite, both frozen in shock over what just happened. Then time reasserts itself and sprite spasms away from the ship with an unearthly crescendo of dissonant tones. It plunges down into the clouds, leaving you standing on your hull with the floating severed piece of tendril.`

Author: Fixed both.

			`	As you re-enter the ship, you notice that your suit is covered in some kind of translucent slime. You scrape off as much as you can into a sealed container, but your pressure suit appears to be permanently stained an iridescent purplish color as a result. You resolve to go find Plume, since he will almost certainly find this interesting.`
				accept
	on accept
		fail "Remnant: Face to Maw 1B"
	on complete
		conversation
			`When you walk into Plume's lab on Aventine you find him buried in some kind of holographic projection of your egg returning mission. You can see that he appears to be watching the video in slow-motion as he taps out notes on his datapad. After a few minutes he looks up and see you standing in the entryway. He quickly pulls himself out of the projection and hits what you assume must be the pause button.`
			`	"Captain <first>! What a pleasant surprise," he trills. "If you are looking for more work, I don't have anything for you right now."`

> 			`	You tell Plume about how you visited Nenia and had an interesting encounter with a void sprite, and offer him the recording and container of slime. He eagerly accepts the datacube and the container from you. He slides the container into some kind of machine that immediately starts whirring, and slots the datacube into his console. Moments later his holographic projection shifts to the new footage. He checks the time stamps, then chants "These will take me a little while to analyze. I will meet you in the spaceport in a few hours to discuss my preliminary findings."`

Author: Fixed.

mission "Remnant: Face to Maw 1B"
	name "Talk to Plume about the void sprites"

> 	description "Plume would be very interested in a report on what happened with the void sprites"

Author: Fixed.

	landing
	to offer
		has "Remnant: Return the Samples 2: done"
	to fail
		or
			has "Remnant: Face to Maw 1: done"
			has "Remnant: Face to Maw 1: active"
	to complete
		has "Remnant: Face to Maw 1: declined"
	source
		system "Nenia"
	destination "Aventine"
	on complete
		conversation
			`When you walk into Plume's lab on Aventine you find him buried in some kind of holographic projection of your egg returning mission. You can see that he appears to be watching the video in slow-motion as he taps out notes on his datapad. After a few minutes he looks up and see you standing in the entryway. He quickly pulls himself out of the projection and hits what you assume must be the pause button.`
			`	"Captain <first>! What a pleasant surprise," he trills. "If you are looking for more work, I don't have anything for you right now."`

> 			`	You tell Plume about how you visited Nenia and had a threatening encounter with a void sprite, and offer him the recording and container with the tentacle. Plume recoils from you in disgust. "After all our work rebuilding trust with the Archon and the void sprites, you go and risk it for... what, exactly? Sightseeing? Thrill of the hunt?" his chant settles into an angry tone. "Leave the recording on the console. I want no part of that tentacle, though."`

Author: Fixed.

			`	As you are leaving Plume's lab a technician pulls you aside. "Our records mention that the tentacles had some interesting properties that were useful in combat. If you are interested, meet me in the spaceport in a few hours."`

mission "Remnant: Face to Maw 2"
	name "Take the slime sample to Viminal"

> 	description "The slime has properties of interest to a team of engineers on <planet>."

Author: FIxed.

	to offer
		has "Remnant: Face to Maw 1: done"
	source "Aventine"
	destination "Viminal"
	cargo "slime sample" 1
	on offer
		conversation

> 			`When Plume finally arrives you can tell from his step that he is excited. He slides onto the bench opposite you and pulls out a datapad. "I admire your bravery, Captain. I don't think I would have the courage to hold my ground against a void sprite like that," he chants respectfully, then changes tone to one of wonder. "In that last moment it almost looked like it was sizing you up or examining you." He shakes his head with an amused expression. "I spend so much time studying non-human creatures I sometimes slip and start anthropomorphizing them." His voice takes on a more militant cadence. "That was probably some kind of territorial display, or possibly some kind of dominance challenge. We don't know enough about void sprite behaviors to speculate further, though."`

Author: Fixed.

			`	"Be that as it may be, the more immediate reward for your curiosity is in this sample," he trills with excitement. "Several of our labs on Viminal have been working on the development of a new generation of ship technology, and this substance contains some chemicals that could be a key component. I have already transmitted the data to them, but they would like to compare with the original. Could you take this sample to them?"`
			choice
				`	"Yes, I can."`
					accept
				`	"I would rather not."`

> 			`	Plume stoically accepts that with a nod "That is your choice. I will arrange for another ship to transport it."`

Author: Fixed.

			decline
	on accept
		event "remnant: penguin"
	on complete
		log "People" "Taely" `One of Taely's main priorities is improving the maneuverability characteristics of Remnant Ships. To this end she has been working on building a new prototype ship based on the void sprites.`
		"remnant taely" ++
		payment 1000000
		conversation
			branch taely
				"remnant taely" > 1

> 			`When you land on Viminal a solidly-built female Remnant in coveralls is waiting for you. "Greetings, Captain <first>. My name is Taely, and I am a Prefect among the Remnant. Above all else, I study motion, and in particular how our starships move."`

Reviewer 6: -ly adverbs should never be [hyphenated.](https://www.merriam-webster.com/words-at-play/6-common-hypercorrections-and-how-to-avoid-them/hyphenating-ly-adverbs)

Author: Fixed.

			goto main
			label taely
			`	When you land on Viminal you spot Taely waiting for you. "Greetings, Captain <first>. It is good to see you again."`
			label main

> 			`	"So, Plume sent us some very interesting data, along with a message to expect you with a sample?" You hand her the container with the slime sample and she holds it up to the light. "Interesting looking stuff," she chants. "Let's go back to my shop." She leads the way through several side passages to an out-of-the-way landing pad connected to a large hanger. Inside, you can see a giant tank filled with a murky fluid, surrounded by equipment and displays. She makes her way over to a large machine with several slots which she slides the container into. As the machine starts to work, she turns back to you.`

Reviewer 6: Again, there's an issue here where Taely continues from the first paragraph into this one, and so there should not be closing quotation marks. Here are two options: We can remove the ending quotation marks from L1807 and L1810, or we can add an introduction to L1812, such as, "Taely straightens up and says, 'So, Plume sent us...'"

["Hanger" should be replaced with "hangar,"](https://www.merriam-webster.com/words-at-play/hangar-vs-hanger) assuming it's referring to a... hangar. (Which sounds like it's the case, since there's also a landing pad.)

I think we should move the last sentence to the next paragraph, so that we don't start the next one with a huge dialogue textwall.

Author: Added `She makes a gesture you are starting to recognize as a sign of curiosity. ` to the start of that line to break up the quotation marks.

Fixed the spelling. 

Moved the sentence.

> 			`	"As the primary example of space-faring life, the void sprites have always been of interest to me. They have numerous advantages that we as atmosphere-bound creatures can only dream of, such as being able to experience space in a much more direct manner. I have been working on a side project to build a new generation of ships that melds these ideas with our current technology, but the data we have on the void sprites is centuries old and incomplete," her chant trails off as she pulls up a display with your original scans on them. "However, your recent explorations provided new data I could use to update my designs. This slime that you retrieved is the final ingredient."`

Reviewer 6: "Her chant trails off" can't be used to end dialogue, unlike "she says."

Author: Switched the comma for a period and capitalized that.

			`	The machine dings, and she looks over the results and then trills excitedly. "Yes, it matches! So it is time to begin the final activation." She strides over to a central console and taps in a series of commands, followed by an iris scan and a blood prick. The industrial hum of the room suddenly ramps up to a crescendo as your nose is assaulted by what you can only describe as the smell of a thunderstorm in a swamp. A few seconds later the noise softens, and the tank starts lowering into the floor as a shape emerges from the liquid.`
			`	The ship before you has some faint traces of the gascraft you have flown to Nenia, along with other elements of Remnant design, but it is clearly something new. Its antennae swivel around, as if taking in its surroundings, and the tentacles along the side twitch and reach out to touch its surroundings. Solidly built tendrils underneath it flex as it lifts itself off its supports and begins to move around. You have a momentary vision of what this ship might look like in the skies of Nenia.`
			scene "scene/penguinscene"

> 			`	Taely's droning chant of technical details finally seeps through your reverie: "... so from a captain's point of view flying the penguin should be very similar to a normal ship, except she flies herself and doesn't need a crew to run her systems." She pauses, and looks at you. "Your curiosity provided the final ingredients to bring her to life. You should be the first to have one. You'll still have to go through the shipyard to register her and pay the resource cost to balance the logistics, but this first one is reserved for you." She pats the nose of the penguin and it makes a wooshing noise. After a moment she starts going over the ship with a scanner, leaving you to contemplate in silence.`

Reviewer 6: Even though Taely's droning is continuing on from somewhere else, "so" should be capitalized as part of normal dialogue rules, since an ellipsis is considered to be a punctuation mark.

We can't call the ship "the penguin" right away, since nobody's named it in dialogue yet.

Author: Capitalized the S.

In regards to calling it the penguin before anyone has named it yet, I did it that way to further the implication that the player (and the pilot) just missed a whole chunk of explanation that presumably included what Taely called it. That way it isn't just that we're *told* that we missed some of what she was saying, we have a concrete instance where the perceptive player can note that "hey, I missed where this got named," thus making the fact that they missed it more personal and not just something they are told.

Reviewer 6: I didn't notice that Taely talked about the penguin in her dialogue, only when the narration said it.

event "remnant: penguin"
	shipyard "Remnant Gascraft"
		add "Penguin"

mission "Remnant: Face to Maw 2B"
	name "Take the tentacle to Chilia's engineers on Caelian"
	description "The tentacle has properties that may be put to use by the engineers on <planet>."
	to offer
		has "Remnant: Face to Maw 1B: done"
	source "Aventine"
	destination "Caelian"
	cargo "tentacle" 1
	on offer
		conversation
			`When the technician arrives in the spaceport he immediately strides over to you. "I checked the records, and my memory was correct. One of our research teams back then discovered that these tentacles could be used as a devastating weapon in close quarters combat."`
			`	He pulls out an image of a strange looking rifle and continues to chant, "the personal weapons lab under Prefect Chilia has a few of the old weapon frames. If you go there, they could turn this tentacle into a potent weapon for you."`
			choice
				`	"That sounds good."`
					accept
				`	"I would rather not."`

> 			`	The technician looks disappointed. "Oh well, enjoy your trophy" he chants.`

Reviewer 6: There should be a comma after "trophy."

Author: Fixed.

			decline
	on complete
		log "People" "Chilia" `As an military prefect, Chilia is responsible for the defence of Remnant space. He can usually be found on the front lines of combat unless required to work on tactics for the Remnant as a whole.`
		payment 250000
		"remnant chilia" ++
		outfit "Void Rifle"
		conversation
			branch chilia
				"remnant chilia" > 1

> 			`You follow the directions provided you and eventually find yourself at a Remnant weapons lab. You enter and are met by security, who ask you to wait. A few minutes later a fit young man emerges from another door. "Greetings, captain <first>," he sings. "I am prefect Chilia. I have the nominal responsibility for the overall defense of Remnant space and military operations." He smiles depreciatingly. "I say 'nominal' as virtually all our activities are military in some way, but usually fall under the purview of those who better understand them."`

Reviewer 6: The introduction should say, "You follow the directions provided *to* you."

I might be wrong here since I've never seen it used as an adverb or adjective, but you might be looking for "deprecate" instead of "depreciate" (as in, "a self-deprecating smile").

I think instead of "military," the activities should be "militaristic." "Our activities are military" sounds very odd to my ears.

"Captain" and "Prefect" should be capitalized, since they are being used as titles here.

Author: Yes, deprecate is what I was going for. That works much better than depreciate. Thanks!

Fixed.

			label main
			`	He leads you to a room that looks like an armory where a team of techs are putting the finishing touches on a rifle stock. "The request from the tech on your behalf caught my attention, as we haven't had the key resources for making void rifles in centuries. Fortunately, we haven't really needed them since we resolved the threats that originally plagued our settlements.`
			choice
				`	"It's called a Void Rifle?"`
				`	"I don't need details."`
					goto end

> 			`	"Yes, Void Rifle. I don't know who discovered it, but someone figured out that if a particularly high charge was poured through the tentacle's nervous system in a specific pattern, it emits a paralyzing cone of energy with a reasonably decent range. I'm not an engineer, so I do not really understand the mechanics of it." He pauses to watch as a tech installs the now-caged tentacle on top of the stock. "Suffice it to say, it is useless for sniping, but its ability to shoot through walls and affect areas makes it excellent for clearing ships and tight quarters. Back then we had a number of threats that made building-to-building clearing operations an unfortunately frequent occurance." His grim tone catches your attention, but he seems unwilling to pursue the topic.`

Reviewer 6: occurrence*

Author: Fixed.

		dialog "The research team quickly offloads the broken jump drive and pays you <payment> before hurrying back to their lab."

mission "Remnant: Face to Maw 1"
	name "Talk to Plume about the void sprites"
	description "Plume would be very interested in a report on what happened with the void sprites."
	landing
	to offer
		has "Remnant: Return the Samples 2: done"
	source
		system "Nenia"
	destination "Aventine"
	on offer
		conversation

> 			`As the swirls of gas close over the <ship> you wonder idly what brought you back here. This time, there is no apologetic researcher asking you to risk your life for research, or to return eggs from where they were stolen. The last glimpse of the Archon drifting overhead as you descend below the cloud deck provides no answers; for now, it is nothing more than a silent, brooding presence in the skies of Nenia. The silent depths of an alien world fold over you, gradually erasing the rest of the galaxy from sight.`

Author: Normally, yes. I was leaving it as implied that it had decloaked momentarily. Changed this to be more specific that the Archon decloaked overhead.

> 			`	As the <ship> levels off at the designated altitude, you strap yourself into a pressure suit. Stepping into the airlock, you grimace at the antiquity of these hard-shell shield systems, and tap the "lower shields" button so you can exit onto the hull. Out on the surface, you admire the misty kaleidescope of colors filtering down through the cloud layers, playing over distant shapes moving in the mists.`

Author: That's alright. I can do that.

> 			`	As the <ship> levels off at the designated altitude, you strap yourself into a pressure suit. Stepping into the airlock, you grimace at the antiquity of these hard-shell shield systems, and tap the "lower shields" button so you can exit onto the hull. Out on the surface, you admire the misty kaleidescope of colors filtering down through the cloud layers, playing over distant shapes moving in the mists.`

Author: Alright, a small piece of foreshadowing added in PR:
https://github.com/endless-sky/endless-sky/pull/4562

> 			`	Before long several void sprites emerge, weaving and dancing among the clouds. They swirl for a few minutes around your cargo bay doors as if waiting for something, then pulse briefly with flickers of light and drop into the clouds below. Another void sprite flutters around the <ship>, poking at things and sliding over them. You quickly think to turn on your video recorder to catch the scene. The void sprite notices your brief movement, and turns towards you with a lunge. As it closes on you, your attention is drawn to what you suspect is its mouth: A zig-zagging crack that peels open like a zipper to reveal an opening resembling an industrial grinder.`

Author: Fixed.

> 			`	Up close, the void sprite is massive: most of its tentacles are easily as thick as your leg, and move with surprising speed and precision. It shifts through the air like a strange kind of fish and you can make out a low vibration you suspect is probably from the air whistling over it as it closes with you. Facing down this rapidly approaching giant, you feel something click in the back of your mind and a wave of paralysis washes over you. In a moment of lucidity moments before the void sprite hits, you recall that your shields are keyed to your equipment: You could activate them to protect yourself from this attack, and they would extend around you. It probably wouldn't be too healthy for the void sprite, though.`

Author: Changed the first two.

In regards to the "click," I meant it in the sense of the Click of a connection forming, as if something just connected to the back of your mind. But if the click is too associated with knowing something, then removing it and leaving it up to the player's imagination works to.. They feel something in the back of their mind and paralysis washes over them.

the second "moments" switched to "just".

In regards to the antique shields being keyed to your equipment, agreed. I'll move that to an earlier mission where the shields will be described a bit more.

Changed the last sentence.

			choice
				`	(Activate shields.)`
				`	(Do not activate shields.)`
					goto holdground

> 			`	Your mind latches onto the idea and you trigger the shields in a rush. The millisecond that follows feels like an eternity: The void sprite is just a few meters away: One tendril reaches out and touches you just as the shield slams into place. You see it strike the tendril and severs it cleanly in the process of re-establishing a sealed barrier. In the same instant, the pressure on your mind vanishes, as if it was simply a dream. You momentarily stare at the immobile void sprite, both frozen in shock over what just happened. Then time reasserts itself and the sprite spasms away from the ship with an unearthly crescendo of dissonant tones. It plunges down into the clouds, leaving you standing on your hull with the floating severed piece of tendril.`

Author: Changed the first sentence as suggested. 

Yeah, that chain of colons was bad. 

Rewrote the chain of colons and tweaked a few things. Not as much as your suggestions. Making things a bit more implied and not pushing reactions on the void sprite. 

In regards to the comment about removing the line about the `pressure on your mind vanishes`... Well, let's just say I'm following your advice about the whole "if you use a gun in the 3rd act, there should be a gun somewhere people might see it in the first act..."

Anyway, how does it read in this revised form?

> 			`	You grab the tendril and store it a sealed container. While the attack spoiled the beautiful moment, at least you have a sample and some recordings that might be of interest to Plume.`

Author: Fair enough. Changed this to "You carefully scoop up the tendril in a container and seal it." 

Not many sticks around in space, but at least this way the player avoids directly touching it.

				flee
			label holdground

> 			`	Despite the paralysis filling your mind, you resolve yourself to face down the void sprite without triggering your shields. Time seems to slow down as the sprite rushes up to you and its leading tentacle reaches out and touches you. As you brace for an impact you realize that the sprite has managed to bring itself to a complete stop and the tentacle has telescoped in on itself slightly. You can see that it is touching you, but you didn't even feel the impact. You and the void sprite stand frozen for a moment, and the mind-numbing paralysis snaps off, replaced by a feeling of weightlessness. Moments later the void sprite pulls back and pulses skyward with an unearthly fountain of sounds, spiraling around the <ship> before disappearing into the swirling clouds.`

Author: I'm trying to hint that the paralysis isn't internal. That the paralysis is something outside the player trying to restrict their actions. That it is, actually, something to be resisted. (and, because the game wouldn't be much fun if the player didn't get to make choices, the player manages to resist enough to make a choice here)

> 			`	Despite the paralysis filling your mind, you resolve yourself to face down the void sprite without triggering your shields. Time seems to slow down as the sprite rushes up to you and its leading tentacle reaches out and touches you. As you brace for an impact you realize that the sprite has managed to bring itself to a complete stop and the tentacle has telescoped in on itself slightly. You can see that it is touching you, but you didn't even feel the impact. You and the void sprite stand frozen for a moment, and the mind-numbing paralysis snaps off, replaced by a feeling of weightlessness. Moments later the void sprite pulls back and pulses skyward with an unearthly fountain of sounds, spiraling around the <ship> before disappearing into the swirling clouds.`

Author: OK, Github is cooperating again. I prefer to have the player resist the paralysis either way. I find that it is more dramatic to hold one's ground against such a charge deliberately, rather than simply give in to paralysis.

> 			`	Despite the paralysis filling your mind, you resolve yourself to face down the void sprite without triggering your shields. Time seems to slow down as the sprite rushes up to you and its leading tentacle reaches out and touches you. As you brace for an impact you realize that the sprite has managed to bring itself to a complete stop and the tentacle has telescoped in on itself slightly. You can see that it is touching you, but you didn't even feel the impact. You and the void sprite stand frozen for a moment, and the mind-numbing paralysis snaps off, replaced by a feeling of weightlessness. Moments later the void sprite pulls back and pulses skyward with an unearthly fountain of sounds, spiraling around the <ship> before disappearing into the swirling clouds.`

Reviewer 2: Then you should really reword it somehow, because it makes no sense to me that whether you do or don't act against the void sprite, you're still somehow acting against a paralysis.

> 			`	Despite the paralysis filling your mind, you resolve yourself to face down the void sprite without triggering your shields. Time seems to slow down as the sprite rushes up to you and its leading tentacle reaches out and touches you. As you brace for an impact you realize that the sprite has managed to bring itself to a complete stop and the tentacle has telescoped in on itself slightly. You can see that it is touching you, but you didn't even feel the impact. You and the void sprite stand frozen for a moment, and the mind-numbing paralysis snaps off, replaced by a feeling of weightlessness. Moments later the void sprite pulls back and pulses skyward with an unearthly fountain of sounds, spiraling around the <ship> before disappearing into the swirling clouds.`

Author: OK. How is:
```	Despite freeing yourself from the paralysis, you resolve yourself to face down the void sprite without triggering your shields. Time seems to slow down as the sprite rushes up to you and its leading tentacle reaches out and touches you. As you brace for an impact you realize that the sprite has managed to bring itself to a complete stop and the tentacle has telescoped in on itself slightly. You can see that it is touching you, but you didn't even feel the impact. You and the void sprite stand frozen for a moment, and the mind-numbing paralysis snaps off, replaced by a feeling of weightlessness. Moments later the void sprite pulls back and pulses skyward with an unearthly fountain of sounds, spiraling around the <ship> before disappearing into the swirling clouds.```

> 			`	Despite the paralysis filling your mind, you resolve yourself to face down the void sprite without triggering your shields. Time seems to slow down as the sprite rushes up to you and its leading tentacle reaches out and touches you. As you brace for an impact you realize that the sprite has managed to bring itself to a complete stop and the tentacle has telescoped in on itself slightly. You can see that it is touching you, but you didn't even feel the impact. You and the void sprite stand frozen for a moment, and the mind-numbing paralysis snaps off, replaced by a feeling of weightlessness. Moments later the void sprite pulls back and pulses skyward with an unearthly fountain of sounds, spiraling around the <ship> before disappearing into the swirling clouds.`

Author: @Amazinite Does that make it more clear that the player is resisting the paralysis, and *then* making a choice as to their action?

			`	As you re-enter the ship, you notice that your suit is covered in some kind of translucent slime. You scrape off as much as you can into a sealed container, but your pressure suit appears to be permanently stained an iridescent purplish color as a result. You resolve to go find Plume, since he will almost certainly find this interesting.`
				accept
	on accept
		fail "Remnant: Face to Maw 1B"
	on complete
		conversation
			`When you walk into Plume's lab on Aventine you find him buried in some kind of holographic projection of your egg returning mission. You can see that he appears to be watching the video in slow-motion as he taps out notes on his datapad. After a few minutes he looks up and see you standing in the entryway. He quickly pulls himself out of the projection and hits what you assume must be the pause button.`
			`	"Captain <first>! What a pleasant surprise," he trills. "If you are looking for more work, I don't have anything for you right now."`

> 			`	You tell Plume about how you visited Nenia and had an interesting encounter with a void sprite, and offer him the recording and container of slime. He eagerly accepts the datacube and the container from you. He slides the container into some kind of machine that immediately starts whirring, and slots the datacube into his console. Moments later his holographic projection shifts to the new footage. He checks the time stamps, then chants, "These will take me a little while to analyze. I will meet you in the spaceport in a few hours to discuss my preliminary findings."`

Author: Changed.

			`	"Captain <first>! What a pleasant surprise," he trills. "If you are looking for more work, I don't have anything for you right now."`
			`	You tell Plume about how you visited Nenia and had a threatening encounter with a void sprite, and offer him the recording and container with the tentacle. Plume recoils from you in disgust. "After all our work rebuilding trust with the Archon and the void sprites, you go and risk it for... what, exactly? Sightseeing? Thrill of the hunt?" His chant settles into an angry tone. "Leave the recording on the console. I want no part of that tentacle, though."`
			`	As you are leaving Plume's lab a technician pulls you aside. "Our records mention that the tentacles had some interesting properties that were useful in combat. If you are interested, meet me in the spaceport in a few hours."`

mission "Remnant: Face to Maw 2"
	name "Take the slime sample to Viminal"
	description "The void sprite slime has properties of interest to a team of engineers on <planet>."
	to offer
		has "Remnant: Face to Maw 1: done"
	source "Aventine"
	destination "Viminal"
	cargo "slime sample" 1
	on offer
		conversation

> 			`When Plume finally arrives you can tell from his step that he is excited. He slides onto the bench opposite you and pulls out a datapad. "I admire your bravery, Captain. I don't think I would have the courage to hold my ground against a void sprite like that," he chants respectfully, then changes tone to one of wonder. "In that last moment it almost looked like it was sizing you up or examining you." He shakes his head with an amused expression. "I spend so much time studying non-human creatures I sometimes slip and start anthropomorphizing them." His voice takes on a more militant cadence. "That was probably some kind of territorial display, or possibly some kind of dominance challenge. We don't know enough about void sprite behaviors to speculate further, though.`

Reviewer 2: `datapad` -> `data pad` as used in "Rim Archaeology 1".
There are two other instances of `datapad` in other Remnant missions that also need changed.

Author: Fixed. 

I put the other datapad->data pad fixes into #4574 so that the fixes are separate from the new content.

			`	"Be that as it may be, the more immediate reward for your curiosity is in this sample," he trills with excitement. "Several of our labs on Viminal have been working on the development of a new generation of ship technology, and this substance contains some chemicals that could be a key component. I have already transmitted the data to them, but they would like to compare with the original. Could you take this sample to them?"`
			choice
				`	"Yes, I can."`
					accept
				`	"I would rather not."`
			`	Plume stoically accepts that with a nod. "That is your choice. I will arrange for another ship to transport it."`
			decline
	on accept
		event "remnant: penguin"
	on complete
		log "People" "Taely" `One of Taely's main priorities is improving the maneuverability characteristics of Remnant Ships. To this end she has been working on building a new prototype ship based on the void sprites.`

> 		"remnant taely" ++

Reviewer 2: Watch out with this. This is the scenario that tehhowch talks about [here](https://github.com/endless-sky/endless-sky/issues/3746#issuecomment-525282390) where a condition is altered and then immediately used in the next conversation.

Author: How should I change it (or does it need to be changed?) The intent is to have one line of dialogue displayed if this is the first time the player has met Taely; and another if it is the second or greater. Which seems to work as intended, unless someone quits right at that moment.

The only way around this that I can think of is moving the log and Taely variable to an invisible mission that triggers and resolves silently after this mission has been marked as "done." And then doing the same for every other mission that has this same branch in it.

I don't really want to expand this PR to do a revamp of other conversations; so if this is the best solution, I would suggest that this PR gets merged with the conversation as-is, and a follow-up PR changes all the conversation branches together.

On the other hand, I'm not too concerned if someone gets two slightly different conversations if they quit right there. I can just imagine the players "What the... Does Taely remember me from the alternate timeline where she already had this conversation with me?"

		payment 1000000
		conversation

> 			branch taely

Reviewer 2: There is WAY too much writing here. I have to scroll to be able to read all of it, which isn't something that should be happening.
![image](https://user-images.githubusercontent.com/17688683/65838752-75def180-e2d4-11e9-8552-5b60575f424c.png)
![image](https://user-images.githubusercontent.com/17688683/65838762-7ecfc300-e2d4-11e9-96e0-345ac50c63a6.png)
Either reduce how much is written, or add a choice or two to break it up.

Author: Yeah, that is rather long.

I'm not too sure how many choices I can add or how to reduce the amount written without cutting into the information provided.

That being said, I have added two breaks: 
Firstly, giving the player a choice when Taely asks them to follow her, the player can choose to catch up in a few minutes, and thus skip the three paragraphs of conversation before the ship reveal.
Secondly, giving the player a chance to respond in the middle of the converstion. Currently the two choices are: 
"So my data is helping to develop new technology?"
"That's good. Any chance of a bonus for me?"

Neither of them goes anywhere, it is just an opportunity to respond. If you have better ideas, I'm open for it.

				"remnant taely" > 1
			`When you land on Viminal a solidly built female Remnant in coveralls is waiting for you. "Greetings, Captain <first>. My name is Taely, and I am a Prefect among the Remnant. Above all else, I study motion, and in particular how our starships move."`

> 			goto main

Reviewer 2: This needs indented one further.

Author: I can't tell exactly what line this is referring to. The `goto` on line 1809?

			scene "scene/penguinscene"
			`	Taely's droning chant of technical details finally seeps through your reverie: "... So from a captain's point of view flying the penguin should be very similar to a normal ship, except she flies herself and doesn't need a crew to run her systems." She pauses, and looks at you. "Your curiosity provided the final ingredients to bring her to life. You should be the first to have one. You'll still have to go through the shipyard to register her and pay the resource cost to balance the logistics, but this first one is reserved for you." She pats the nose of the penguin and it makes a wooshing noise. After a moment she starts going over the ship with a scanner, leaving you to contemplate in silence.`

event "remnant: penguin"
	shipyard "Remnant Gascraft"

> 		add "Penguin"

Reviewer 2: Remove `add`.
```
event "dreadnoughts for sale"
	shipyard "Southbound Advanced"
		"Dreadnought"
```

Author: `add` removed.

mission "Remnant: Face to Maw 2B"
	name "Take the tentacle to Chilia's engineers on Caelian"
	description "The tentacle has properties that may be put to use by the engineers on <planet>."
	to offer
		has "Remnant: Face to Maw 1B: done"
	source "Aventine"
	destination "Caelian"
	cargo "tentacle" 1
	on offer
		conversation

> 			`When the technician arrives in the spaceport he immediately strides over to you. "I checked the records, and my memory was correct. One of our research teams back then discovered that these tentacles could be used as a devastating weapon in close quarters combat."`

Reviewer 2: `back then` -> `centuries ago`

Author: Changed.

> 			`	He pulls out an image of a strange looking rifle and continues to chant, "the personal weapons lab under Prefect Chilia has a few of the old weapon frames. If you go there, they could turn this tentacle into a potent weapon for you."`

Reviewer 2: `the` -> `The`.

Author: Changed.

			choice
				`	"That sounds good."`

> 					accept

Reviewer 2: The `<destination>` isn't given anywhere before accepting.

Author: Added a bit to describe where it is.

				`	"I would rather not."`

> 			`	The technician looks disappointed. "Oh well, enjoy your trophy," he chants.`

Reviewer 2: At this point I feel like highlighting #3636. Not everyone needs to always "chant" or "trill." They can just "say" sometimes.

Author: Removed.

				goto main
			label chilia
			`You follow the directions provided you and eventually find yourself at a Remnant weapons lab. You enter and are met by security, who ask you to wait. A few minutes later a fit young man emerges from another door. "Greetings, captain <first>," he sings. "We meet once again." You immediately recognize the voice as being the man who convinced you to provide the blood sample and get to know the Remnant. "We didn't get a proper introduction last time, but you have probably figured out that I have the nominal responsibility for the overall defense of Remnant space and military operations." He smiles depreciatingly, "I say 'nominal' as virtually all our activities are military in some way, but usually fall under the purview of those who better understand them."`
			label main

> 			`	He leads you to a room that looks like an armory where a team of techs are putting the finishing touches on a rifle stock. "The request from the tech on your behalf caught my attention, as we haven't had the key resources for making void rifles in centuries. Fortunately, we haven't really needed them since we resolved the threats that originally plagued our settlements.`

Reviewer 2: `tech(s)` -> `technician(s)` 
Inconsistent capitalization of void rifles. In this paragraph it's not capitalized, but in the following choice and paragraph it is.

Author: Fixed.

			choice
				`	"It's called a Void Rifle?"`
				`	"I don't need details."`
					goto end

> 			`	"Yes, Void Rifle. I don't know who discovered it, but someone figured out that if a particularly high charge was poured through the tentacle's nervous system in a specific pattern, it emits a paralyzing cone of energy with a reasonably decent range. I'm not an engineer, so I do not really understand the mechanics of it." He pauses to watch as a tech installs the now-caged tentacle on top of the stock. "Suffice it to say, it is useless for sniping, but its ability to shoot through walls and affect areas makes it excellent for clearing ships and tight quarters. Back then we had a number of threats that made building-to-building clearing operations an unfortunately frequent occurrence." His grim tone catches your attention, but he seems unwilling to pursue the topic.`

Reviewer 2: I feel like a biologist would have more understanding of it than an engineer. :)
`Back then we had a number of threats...` I'd say remove this part. Chilia already referenced a vague threat earlier. Continually referencing a vague threat without defining it just sounds like bad story telling to me.

Author: Changed.

			label end

> 			`	"All right, enough history." He turns to the techs who are finishing the assembly. "The tentacle has been treated to resist decay and the stock mechanisms will provide basic maintenance routines for as long as you charge it regularly." He strides over, picks up the completed void rifle, and then hands it to you. "You are holding a piece of our history, Captain. Use it well!"`

Reviewer 2: Again, `void rifle` is used here here but `Void Rifle` is used above. Since `void sprite` isn't capitalized, I'd do the same with `void rifle`.

Author: Change "Void Rifle" to "void rifle" everywhere except the name of the outfit

		dialog "The research team quickly offloads the broken jump drive and pays you <payment> before hurrying back to their lab."

mission "Remnant: Face to Maw 1"
	name "Talk to Plume about the void sprites"
	description "Plume would be very interested in a report on what happened with the void sprites."
	landing
	to offer
		has "Remnant: Return the Samples 2: done"
	source
		system "Nenia"
	destination "Aventine"
	on offer
		conversation

> 			`As the swirls of gas close over the <ship> you wonder idly what brought you back here. This time, there is no apologetic researcher asking you to risk your life for research, or to return eggs from where they were stolen. The last glimpse of the Archon decloaking overhead as you descend below the cloud deck provides no answers; for now, it is nothing more than a silent, brooding presence in the skies of Nenia. The silent depths of an alien world fold over you, gradually erasing the rest of the galaxy from sight.`

Author: OK. Changed.

> 			`As the swirls of gas close over the <ship> you wonder idly what brought you back here. This time, there is no apologetic researcher asking you to risk your life for research, or to return eggs from where they were stolen. The last glimpse of the Archon decloaking overhead as you descend below the cloud deck provides no answers; for now, it is nothing more than a silent, brooding presence in the skies of Nenia. The silent depths of an alien world fold over you, gradually erasing the rest of the galaxy from sight.`

Author: Fixed again.

			`	As the <ship> levels off at the designated altitude, you strap yourself into a pressure suit. Stepping into the airlock, you grimace at the antiquity of these hard-shell shield systems, and tap the "lower shields" button so you can exit onto the hull. Out on the surface, you admire the misty kaleidescope of colors filtering down through the cloud layers, playing over distant shapes moving in the mists.`

> 			`	Before long several void sprites emerge, weaving and dancing among the clouds. They swirl for a few minutes around your cargo bay doors as if waiting for something, then pulse briefly with flickers of light and drop into the clouds below. Another void sprite flutters around the <ship>, poking at things and sliding over them with its tentacles. You quickly think to turn on your video recorder to catch the scene. The void sprite notices your brief movement, and turns towards you with a lunge. As it closes on you, your attention is drawn to what you suspect is its mouth: a zig-zagging crack that peels open like a zipper to reveal an opening resembling an industrial grinder.`

Author: OK. Changed.

> 			`	Up close, the void sprite is massive: most of its tentacles are easily as thick as your leg, and move with surprising speed and precision. It shifts through the air like a strange deep sea fish in water and you can make out a low vibration you suspect is from the air whistling over it as it closes with you. Facing down this rapidly approaching giant, you feel something in the back of your mind and a wave of paralysis washes over you. In a moment of lucidity just before the void sprite hits, you recall that your shields are keyed to your equipment: You could activate them to stop the void sprite from getting to you, but contact with the ancient shields probably wouldn't be healthy for the void sprite.`

Author: Split, changed, replaced with a semicolon. Thanks for the feedback!

			choice
				`	(Activate shields.)`
				`	(Do not activate shields.)`
					goto holdground

> 			`	You trigger the shields in a rush. The millisecond that follows feels like an eternity: The void sprite races forward with one tendril reaching out to touch you just as the shield slams into place. You see the energy strike the tendril and sever it cleanly in the process of re-establishing a sealed barrier. In the same instant, the pressure on your mind vanishes as if it was simply a dream. You momentarily stare at the immobile void sprite, frozen in shock over what just happened. Then time reasserts itself and the sprite spasms away from the ship with an unearthly crescendo of dissonant tones. It plunges down into the clouds, leaving you standing on your hull with the floating severed piece of tendril.`

Author: I like my vocabulary. But sure, changed.

		dialog "The research team quickly offloads the broken jump drive and pays you <payment> before hurrying back to their lab."

mission "Remnant: Face to Maw 1"
	name "Talk to Plume about the void sprites"
	description "Plume would be very interested in a report on what happened with the void sprites."
	landing
	to offer

> 		has "Remnant: Return the Samples 2: done"

Reviewer 7: Would it make sense to offer this mission only a few weeks after the sample mission chain? The first lines of dialog feel kinda weird if the player just returned to Nenia immediately

Author: Hmmm. You're right. It would fit better if there was at least a short delay. Might also keep players guessing about where else they should try heading back to in the future. I guess I'll need to make an event timer.

Author: OK, event timer set for 14 days added.

	source
		system "Nenia"
	destination "Aventine"
	on offer
		conversation
			`As the swirls of gas close over the <ship> you wonder idly what brought you back here. This time, there is no apologetic researcher asking you to risk your life for research, or to return eggs from where they were stolen. You catch a glimpse of the Archon materializing above the clouds as it decloaks, letting you know that you're being watched, but saying nothing; for now, it is nothing more than a silent, brooding presence in the skies of Nenia. The silent depths of an alien world fold over you, gradually erasing the rest of the galaxy from sight.`
			`	As the <ship> levels off at the designated altitude, you strap yourself into a pressure suit. Stepping into the airlock, you grimace at the antiquity of these hard-shell shield systems, and tap the "lower shields" button so you can exit onto the hull. Out on the surface, you admire the misty kaleidescope of colors filtering down through the cloud layers, playing over distant shapes moving in the mists.`

> 			`	Before long several void sprites emerge, weaving and dancing among the clouds. They swirl for a few minutes around your cargo bay doors as if waiting for something, then pulse briefly with flickers of light and drop into the clouds below. Another void sprite flutters around the <ship>, poking at the hull and sliding over it with its tentacles. You quickly think to turn on your video recorder to catch the scene. The void sprite notices your brief movement, and turns towards you with a lunge. As it closes on you, your attention is drawn to what you suspect is its mouth: A zig-zagging crack that peels open like a zipper to reveal an opening that resembles an industrial grinder.`

Author: I can't tell what changed in your commit suggestion there. GitHub isn't highlighting anything for me.

> 			`	Before long several void sprites emerge, weaving and dancing among the clouds. They swirl for a few minutes around your cargo bay doors as if waiting for something, then pulse briefly with flickers of light and drop into the clouds below. Another void sprite flutters around the <ship>, poking at the hull and sliding over it with its tentacles. You quickly think to turn on your video recorder to catch the scene. The void sprite notices your brief movement, and turns towards you with a lunge. As it closes on you, your attention is drawn to what you suspect is its mouth: A zig-zagging crack that peels open like a zipper to reveal an opening that resembles an industrial grinder.`

Reviewer 8: `mouth: A zig-zagging` -> `mouth: a zig-zagging`

> 			`	Before long several void sprites emerge, weaving and dancing among the clouds. They swirl for a few minutes around your cargo bay doors as if waiting for something, then pulse briefly with flickers of light and drop into the clouds below. Another void sprite flutters around the <ship>, poking at the hull and sliding over it with its tentacles. You quickly think to turn on your video recorder to catch the scene. The void sprite notices your brief movement, and turns towards you with a lunge. As it closes on you, your attention is drawn to what you suspect is its mouth: A zig-zagging crack that peels open like a zipper to reveal an opening that resembles an industrial grinder.`

Author: Ah, OK. Thanks for pointing that out.

			`	Up close, the void sprite is massive. Most of its tentacles are easily as thick as your leg, and move with surprising speed and precision. It shifts through the air like a strange deep sea fish in water. You can make out a low vibration you suspect is from the air whistling over the void sprite as it closes with you. As the giant creature rapidly approaches, you can feel a heavy paralysis weigh down on you. Suddenly, you recall that your shields are keyed to your equipment; you could activate them to protect yourself and they would extend around you. It probably wouldn't be too healthy for the void sprite, though.`
			choice

> 				`	(Activate shields.)`

Reviewer 2: ```suggestion
				`	(Activate the shields.)`
```

Author: Fixed. Thanks!

> 				`	(Do not activate shields.)`

Author: Sure

			`	As you re-enter the ship, you notice that your suit is covered in some kind of translucent slime. You scrape off as much as you can into a sealed container, but your pressure suit appears to be permanently stained an iridescent purplish color as a result. You resolve to go find Plume, since he will almost certainly find this interesting.`
				accept
	on accept
		fail "Remnant: Face to Maw 1B"
	on complete
		conversation

> 			`When you walk into Plume's lab on Aventine you find him buried in some kind of holographic projection of your egg returning mission. You can see that he appears to be watching the video in slow-motion as he taps out notes on his data pad. After a few minutes he looks up and see you standing in the entryway. He quickly pulls himself out of the projection and hits what you assume must be the pause button.`

Reviewer 7: ```suggestion
			`When you walk into Plume's lab on Aventine you find him buried in some kind of holographic projection of your egg returning mission. You can see that he appears to be watching the video in slow-motion as he taps out notes on his data pad. After a few minutes he looks up and sees you standing in the entryway. He quickly pulls himself out of the projection and hits what you assume must be the pause button.`
```
alternatively, `notices` seems like a good fit.

			`	"Captain <first>! What a pleasant surprise," he trills. "If you are looking for more work, I don't have anything for you right now."`

> 			`	You tell Plume about how you visited Nenia and had an interesting encounter with a void sprite, and offer him the recording and container of slime. He eagerly accepts the data chip and the container from you. He slides the container into some kind of machine that immediately starts whirring, and slots the data chip into his console. Moments later his holographic projection shifts to the new footage. He checks the time stamps, then chants, "These will take me a little while to analyze. I will meet you in the spaceport in a few hours to discuss my preliminary findings."`

Reviewer 7: Two sentences starting with `He` in quick succession, followed by another one. That might be a bit clunky to read.

Author: I rewrote that middle sentence a bit to break things up. How's this?
`He eagerly accepts the data chip and the container from you. Sliding the container into some kind of machine, it immediately starts to whir as he slots the data chip into his console.`

Reviewer 7: much better, yes.

Author: Great!

mission "Remnant: Face to Maw 1B"
	name "Talk to Plume about the void sprites"
	description "Plume would be very interested in a report on what happened with the void sprites."
	landing
	to offer
		has "Remnant: Return the Samples 2: done"
	to fail
		or
			has "Remnant: Face to Maw 1: done"
			has "Remnant: Face to Maw 1: active"
	to complete
		has "Remnant: Face to Maw 1: declined"
	source
		system "Nenia"
	destination "Aventine"
	on complete
		conversation

> 			`When you walk into Plume's lab on Aventine you find him buried in some kind of holographic projection of your egg returning mission. You can see that he appears to be watching the video in slow-motion as he taps out notes on his data pad. After a few minutes he looks up and see you standing in the entryway. He quickly pulls himself out of the projection and hits what you assume must be the pause button.`

Reviewer 7: ```suggestion
			`When you walk into Plume's lab on Aventine you find him buried in some kind of holographic projection of your egg returning mission. You can see that he appears to be watching the video in slow-motion as he taps out notes on his data pad. After a few minutes he looks up and sees you standing in the entryway. He quickly pulls himself out of the projection and hits what you assume must be the pause button.`
```
alternatively, `notices` seems like a good fit.

			`	"Captain <first>! What a pleasant surprise," he trills. "If you are looking for more work, I don't have anything for you right now."`
			`	You tell Plume about how you visited Nenia and had a threatening encounter with a void sprite, and offer him the recording and container with the tentacle. Plume recoils from you in disgust. "After all our work rebuilding trust with the Archon and the void sprites, you go and risk it for... what, exactly? Sightseeing? Thrill of the hunt?" His chant settles into an angry tone. "Leave the recording on the console. I want no part of that tentacle, though."`
			`	As you are leaving Plume's lab a technician pulls you aside. "Our records mention that the tentacles had some interesting properties that were useful in combat. If you are interested, meet me in the spaceport in a few hours."`

mission "Remnant: Face to Maw 2"
	name "Take the slime sample to Viminal"
	description "The void sprite slime has properties of interest to a team of engineers on <planet>."
	to offer
		has "Remnant: Face to Maw 1: done"
	source "Aventine"
	destination "Viminal"
	cargo "slime sample" 1
	on offer
		conversation

> 			`When Plume finally arrives you can tell from his step that he is excited. He slides onto the bench opposite you and pulls out a data pad. "I admire your bravery, Captain. I don't think I would have the courage to hold my ground against a void sprite like that," he chants respectfully, then changes tone to one of wonder. "In that last moment it almost looked like it was sizing you up or examining you." He shakes his head with an amused expression. "I spend so much time studying non-human creatures I sometimes slip and start anthropomorphizing them." His voice takes on a more militant cadence. "That was probably some kind of territorial display, or possibly some kind of dominance challenge. We don't know enough about void sprite behaviors to speculate further, though.`

Reviewer 7: ```suggestion
			`When Plume finally arrives you can tell from his step that he is excited. He slides onto the bench opposite to you and pulls out a data pad. "I admire your bravery, Captain. I don't think I would have the courage to hold my ground against a void sprite like that," he chants respectfully, then changes tone to one of wonder. "In that last moment it almost looked like it was sizing you up or examining you." He shakes his head with an amused expression. "I spend so much time studying non-human creatures I sometimes slip and start anthropomorphizing them." His voice takes on a more militant cadence. "That was probably some kind of territorial display, or possibly some kind of dominance challenge. We don't know enough about void sprite behaviors to speculate further, though.`
```
in case it's hard to see, the change is to `opposite to you`, second sentence.

Author: Thanks. Github doesn't always highlight the change.

> 			`When Plume finally arrives you can tell from his step that he is excited. He slides onto the bench opposite you and pulls out a data pad. "I admire your bravery, Captain. I don't think I would have the courage to hold my ground against a void sprite like that," he chants respectfully, then changes tone to one of wonder. "In that last moment it almost looked like it was sizing you up or examining you." He shakes his head with an amused expression. "I spend so much time studying non-human creatures I sometimes slip and start anthropomorphizing them." His voice takes on a more militant cadence. "That was probably some kind of territorial display, or possibly some kind of dominance challenge. We don't know enough about void sprite behaviors to speculate further, though.`

Reviewer 7: ```suggestion
			`When Plume finally arrives you can tell from his step that he is excited. He slides onto the bench opposite you and pulls out a data pad. "I admire your bravery, Captain. I don't think I would have had the courage to hold my ground against a void sprite like that," he chants respectfully, then changes tone to one of wonder. "In that last moment it almost looked like it was sizing you up or examining you." He shakes his head with an amused expression. "I spend so much time studying non-human creatures I sometimes slip and start anthropomorphizing them." His voice takes on a more militant cadence. "That was probably some kind of territorial display, or possibly some kind of dominance challenge. We don't know enough about void sprite behaviors to speculate further, though.`
```
in case it's hard to see, the change is to `would have had`, 4th sentence.

Author: Fixed. Thanks!

> 			`When Plume finally arrives you can tell from his step that he is excited. He slides onto the bench opposite you and pulls out a data pad. "I admire your bravery, Captain. I don't think I would have the courage to hold my ground against a void sprite like that," he chants respectfully, then changes tone to one of wonder. "In that last moment it almost looked like it was sizing you up or examining you." He shakes his head with an amused expression. "I spend so much time studying non-human creatures I sometimes slip and start anthropomorphizing them." His voice takes on a more militant cadence. "That was probably some kind of territorial display, or possibly some kind of dominance challenge. We don't know enough about void sprite behaviors to speculate further, though.`

Reviewer 7: ```suggestion
			`When Plume finally arrives you can tell from his step that he is excited. He slides onto the bench opposite you and pulls out a data pad. "I admire your bravery, Captain. I don't think I would have the courage to hold my ground against a void sprite like that," he chants respectfully, then changes tone to one of wonder. "In that last moment it almost looked like it was sizing you up or examining you." He shakes his head with an amused expression. "I spend so much time studying non-human creatures I sometimes slip and start anthropomorphizing them." His voice takes on a more militant cadence. "That was probably some kind of territorial display, or possibly some kind of dominance challenge. We don't know enough about void sprite behavior to speculate further, though.`
```
in case it's hard to see, the change is to `behavior`, last sentence.

Author: Fixed. Thanks!

			choice
				`	"Yes, I can."`
					accept
				`	"I would rather not."`
			`	Plume stoically accepts that with a nod. "That is your choice. I will arrange for another ship to transport it."`
			decline
	on accept
		event "remnant: penguin"
	on complete
		log "People" "Taely" `One of Taely's main priorities is improving the maneuverability characteristics of Remnant Ships. To this end she has been working on building a new prototype ship based on the void sprites.`
		"remnant taely" ++
		payment 1000000
		conversation
			branch taely
				"remnant taely" > 1
			`When you land on Viminal a solidly built female Remnant in coveralls is waiting for you. "Greetings, Captain <first>. My name is Taely, and I am a Prefect among the Remnant. Above all else, I study motion, and in particular how our starships move."`
				goto main
			label taely
			`	When you land on Viminal you spot Taely waiting for you. "Greetings, Captain <first>. It is good to see you again."`
			label main

> 			`	She makes a gesture you are starting to recognize as a sign of curiosity. "So, Plume sent us some very interesting data, along with a message to expect you with a sample?" You hand her the container with the slime sample and she holds it up to the light. "Interesting looking stuff," she chants. "Let's go back to my workshop."`

Reviewer 7: i don't know, "stuff" just sounds too common for a remnant. i would expect something along the lines of "substance", "component", etc.

Author: I agree. Changed.

> 			`	She makes a gesture you are starting to recognize as a sign of curiosity. "So, Plume sent us some very interesting data, along with a message to expect you with a sample?" You hand her the container with the slime sample and she holds it up to the light. "Interesting looking stuff," she chants. "Let's go back to my workshop." `

Reviewer 2: Extra space here after the final quotation mark.

Author: Fixed.

			choice
				`	(Ask if you can catch up in a few minutes)`

> 				`	(Follow her back to her workshop)`

Reviewer 2: Flip these choices (so that the positive one comes first) and make them actual dialog instead of actions. (Even if they were suppose to be actions, you're missing your punctuation.)

Author: Changed.

> 				`	(Ask if you can catch up in a few minutes)`

Reviewer 2: These two choices still aren't dialog.

Author: I didn't think choices had to be dialog, do they?

Reviewer 2: It makes no sense for these to be action choices though. especially when one of them is an action to say something; why not just have the choice be what you say?

Author: OK, changed. I prefered it as an action, because then the player can imagine whatever sort of dialog they want instead of whatever I put in their mouth. But if you want actual dialog, then it is done.

> 			`	"Sure, just don't take too long. My project has reached its final stage."

Reviewer 2: Missing closing backtick.

Author: Fixed.

> 			`	You take your time to get to the workshop, admiring the view along the way. When you finally arrive at the workshop you can hear strange noises coming form inside, accompanied by the smell of a thunderstorm in a swamp. You step into the workshop and come to a stop to stare at the thing in front of you.`

Reviewer 7: ```suggestion
			`	You take your time to get to the workshop, admiring the view along the way. When you finally arrive at the workshop you can hear strange noises coming from inside, accompanied by the smell of a thunderstorm in a swamp. You step into the workshop and come to a stop to stare at the thing in front of you.`
```
also: `from the inside`?

				`	"So my data is helping to develop new technology?"`
				`	"That's good. Any chance of a bonus for me?"`
			`	The machine dings, and she looks over the results and then trills excitedly. "Yes, it matches! So it is time to begin the final activation." She strides over to a central console and taps in a series of commands, followed by an iris scan and a blood prick. The industrial hum of the room suddenly ramps up to a crescendo as your nose is assaulted by what you can only describe as the smell of a thunderstorm in a swamp. A few seconds later the noise softens, and the tank starts lowering into the floor as a shape emerges from the liquid.`
			label skip
			`	The ship before you has some faint traces of the gascraft you have flown to Nenia, along with other elements of Remnant design, but it is clearly something new. Its antennae swivel around, as if taking in its environment, and the tentacles along the side twitch and reach out to touch its surroundings. Solidly built tendrils underneath it flex as it lifts itself off its supports and begins to move around. You have a momentary vision of what this ship might look like in the skies of Nenia.`
			scene "scene/penguinscene"

> 			`	Taely's droning chant of technical details finally seeps through your reverie: "... So from a captain's point of view flying the penguin should be very similar to a normal ship, except she flies herself and doesn't need a crew to run her systems." She pauses, and looks at you. "Your curiosity provided the final ingredients to bring her to life. You should be the first to have one. You'll still have to go through the shipyard to register her and pay the resource cost to balance the logistics, but this first one is reserved for you." She pats the nose of the penguin and it makes a wooshing noise. After a moment she starts going over the ship with a scanner, leaving you to contemplate in silence.`

Reviewer 2: Capitalize Penguin.

Author: Capitalized.

			choice
				`	"That sounds good."`
					accept
				`	"I would rather not."`
			`	The technician looks disappointed. "Oh well, enjoy your trophy."`
			decline
	on complete
		log "People" "Chilia" `As an military prefect, Chilia is responsible for the defence of Remnant space. He can usually be found on the front lines of combat unless required to work on tactics for the Remnant as a whole.`
		payment 250000
		"remnant chilia" ++
		outfit "Void Rifle"
		conversation
			branch chilia
				"remnant chilia" > 1
			`You follow the directions provided to you and eventually find yourself at a Remnant weapons lab. You enter and are met by security, who ask you to wait. A few minutes later a fit young man emerges from another door. "Greetings, Captain <first>," he sings. "I am Prefect Chilia. I have the nominal responsibility for the overall defense of Remnant space and military operations." He smiles depreciatingly. "I say 'nominal' as virtually all our activities are militaristic in some way, but usually fall under the purview of those who better understand them."`
				goto main
			label chilia

> 			`You follow the directions provided you and eventually find yourself at a Remnant weapons lab. You enter and are met by security, who ask you to wait. A few minutes later a fit young man emerges from another door. "Greetings, captain <first>," he sings. "We meet once again." You immediately recognize the voice as being the man who convinced you to provide the blood sample and get to know the Remnant. "We didn't get a proper introduction last time, but you have probably figured out that I have the nominal responsibility for the overall defense of Remnant space and military operations." He smiles depreciatingly, "I say 'nominal' as virtually all our activities are military in some way, but usually fall under the purview of those who better understand them."`

Reviewer 7: ```suggestion
			`You follow the directions provided to you and eventually find yourself at a Remnant weapons lab. You enter and are met by security, who ask you to wait. A few minutes later a fit young man emerges from another door. "Greetings, captain <first>," he sings. "We meet once again." You immediately recognize the voice as being the man who convinced you to provide the blood sample and get to know the Remnant. "We didn't get a proper introduction last time, but you have probably figured out that I have the nominal responsibility for the overall defense of Remnant space and military operations." He smiles depreciatingly, "I say 'nominal' as virtually all our activities are military in some way, but usually fall under the purview of those who better understand them."`
```

			label main

> 			`	He leads you to a room that looks like an armory where a team of technicians are putting the finishing touches on a rifle stock. "The request from the tech on your behalf caught my attention, as we haven't had the key resources for making void rifles in centuries. Fortunately, we haven't really needed them since we resolved the threats that originally plagued our settlements.`

Reviewer 7: Similarly to my earlier comment, would a remnant really say "the tech"?

Author: No, not really. Fixed. Thanks!

			choice
				`	"It's called a void rifle?"`
				`	"I don't need details."`
					goto end

> 			`	"Yes, void rifle. I don't know who discovered it, but someone figured out that if a particularly high charge was poured through the tentacle's nervous system in a specific pattern, it emits a paralyzing cone of energy with a reasonably decent range. I'm neither an engineer or a biologist, so I do not really understand the mechanics of it." He pauses to watch as a tech installs the now-caged tentacle on top of the stock. "Suffice it to say, it is useless for sniping, but its ability to shoot through walls and affect areas makes it excellent for sweeping ships and building-to-building clearing operations.`

Reviewer 7: see above, "tech"

Author: Thanks for spotting these. Fixed.

		dialog "The research team quickly offloads the broken jump drive and pays you <payment> before hurrying back to their lab."

mission "Remnant: Face to Maw 1"
	name "Talk to Plume about the void sprites"
	description "Plume would be very interested in a report on what happened with the void sprites."
	landing
	to offer
		has "remnant: return the samples timer"
	source
		system "Nenia"
	destination "Aventine"
	on offer
		conversation
			`As the swirls of gas close over the <ship> you wonder idly what brought you back here. This time, there is no apologetic researcher asking you to risk your life for research, or to return eggs from where they were stolen. You catch a glimpse of the Archon materializing above the clouds as it decloaks, letting you know that you're being watched, but saying nothing; for now, it is nothing more than a silent, brooding presence in the skies of Nenia. The silent depths of an alien world fold over you, gradually erasing the rest of the galaxy from sight.`

> 			`	As the <ship> levels off at the designated altitude, you strap yourself into a pressure suit. Stepping into the airlock, you grimace at the antiquity of these "hard-shell" shield systems, and lower the shields so that you can exit onto the hull. Out on the surface, you admire the misty kaleidescope of colors filtering down through the cloud layers, playing over distant shapes moving in the mists.`

Reviewer 7: ```suggestion
			`	As the <ship> levels off at the designated altitude, you strap yourself into a pressure suit. Stepping into the airlock, you grimace at the antiquity of these "hard-shell" shield systems, and lower the shields so that you can exit onto the hull. Out on the surface, you admire the misty kaleidoscope of colors filtering down through the cloud layers, playing over distant shapes moving in the mists.`
```

			`	Before long several void sprites emerge, weaving and dancing among the clouds. They swirl for a few minutes around your cargo bay doors as if waiting for something, then pulse briefly with flickers of light and drop into the clouds below. Another void sprite flutters around the <ship>, poking at the hull and sliding over it with its tentacles. You quickly think to turn on your video recorder to catch the scene. The void sprite notices your brief movement, and turns towards you with a lunge. As it closes on you, your attention is drawn to what you suspect is its mouth: a zig-zagging crack that peels open like a zipper to reveal an opening that resembles an industrial grinder.`

> 			`	Up close, the void sprite is massive. Most of its tentacles are easily as thick as your leg, and move with surprising speed and precision. It shifts through the air like a strange deep sea fish in water. You can make out a low vibration you suspect is from the air whistling over the void sprite as it closes with you. As the giant creature rapidly approaches, you can feel a heavy paralysis weigh down on you. Suddenly, you recall that your shields are keyed to your equipment; you could activate them to protect yourself and they would extend around you. It probably wouldn't be too healthy for the void sprite, though.`

Reviewer 7: ```suggestion
			`	Up close, the void sprite is massive. Most of its tentacles are easily as thick as your leg, and move with surprising speed and precision. It shifts through the air like a strange deep-sea fish in water. You can make out a low vibration you suspect is from the air whistling over the void sprite as it closes with you. As the giant creature rapidly approaches, you can feel a heavy paralysis weigh down on you. Suddenly, you recall that your shields are keyed to your equipment; you could activate them to protect yourself and they would extend around you. It probably wouldn't be too healthy for the void sprite, though.`
```
`deep-sea`, according to my spell checker

			choice
				`	(Activate the shields.)`
				`	(Do not activate the shields.)`
					goto holdground
			`	You push against the paralysis and trigger the shields in a rush. The millisecond that follows feels like an eternity. The void sprite is just a few meters away, one tendril reaches out and touches you just as the shield slams into place. The tendril becomes caught on both sides of the shield, cleaving it in two. In the same instant, the pressure on your mind vanishes as if it was simply a dream. You momentarily stare at the immobile void sprite, frozen in shock over what just happened. Then time reasserts itself and the sprite spasms away from the ship with an unearthly crescendo of dissonant tones. It plunges down into the clouds, leaving you standing on your hull with the floating severed piece of tendril.`
			`	You carefully scoop up the tendril in a container and seal it. While the attack spoiled the beautiful moment, at least you have a sample and some recordings that might be of interest to Plume.`
				flee
			label holdground

> 			`	The feeling of paralysis leaves your body as the void sprite is only a few meters away, but you resolve yourself to face down the creature without triggering your shields. Time seems to slow down as the sprite rushes up to you and its leading tentacle reaches out to touch you. As you brace for an impact you realize that the sprite has managed to bring itself to a complete stop and the tentacle has telescoped in on itself slightly. You can see that it is touching you, but you didn't even feel the impact. You and the void sprite stand frozen for a moment, and the force trying to paralyze you snaps off, replaced by a feeling of weightlessness. Moments later the void sprite pulls back and pulses skyward with an unearthly fountain of sounds, spiraling around the <ship> before disappearing into the swirling clouds.`

Reviewer 7: `You and the void sprite stand frozen for a moment`
not to nitpick, but technically they're floating..

Author: Well, I picture the captain standing on the hull with magnetic boots or something, but yeah, the void sprite would be floating. `you and the void sprite float frozen for a moment.` doesn't really have the same ring to it.

Reviewer 7: fair enough

			`	As you re-enter the ship, you notice that your suit is covered in some kind of translucent slime. You scrape off as much as you can into a sealed container, but your pressure suit appears to be permanently stained an iridescent purplish color as a result. You resolve to go find Plume, since he will almost certainly find this interesting.`
				accept
	on accept
		fail "Remnant: Face to Maw 1B"
	on complete
		conversation
			`When you walk into Plume's lab on Aventine you find him buried in some kind of holographic projection of your egg returning mission. You can see that he appears to be watching the video in slow-motion as he taps out notes on his data pad. After a few minutes he looks up and sees you standing in the entryway. He quickly pulls himself out of the projection and hits what you assume must be the pause button.`
			`	"Captain <first>! What a pleasant surprise," he trills. "If you are looking for more work, I don't have anything for you right now."`

> 			`	You tell Plume about how you visited Nenia and had an interesting encounter with a void sprite, and offer him the recording and container of slime. He eagerly accepts the data chip and the container from you. Sliding the container into some kind of machine, it immediately starts to whir as he slots the data chip into his console. Moments later his holographic projection shifts to the new footage. He checks the time stamps, then chants, "These will take me a little while to analyze. I will meet you in the spaceport in a few hours to discuss my preliminary findings."`

Reviewer 7: ```suggestion
			`	You tell Plume about how you visited Nenia and had an interesting encounter with a void sprite, and offer him the recording and container of slime. He eagerly accepts the data chip and the container from you. Sliding the container into some kind of machine, it immediately starts to whir as he slots the data chip into his console. Moments later his holographic projection shifts to the new footage. He checks the timestamps, then chants, "These will take me a little while to analyze. I will meet you in the spaceport in a few hours to discuss my preliminary findings."`
```
`timestamps`

			decline
	on accept
		event "remnant: penguin"
	on complete
		log "People" "Taely" `One of Taely's main priorities is improving the maneuverability characteristics of Remnant Ships. To this end she has been working on building a new prototype ship based on the void sprites.`
		"remnant taely" ++
		payment 1000000
		conversation
			branch taely
				"remnant taely" > 1
			`When you land on Viminal a solidly built female Remnant in coveralls is waiting for you. "Greetings, Captain <first>. My name is Taely, and I am a Prefect among the Remnant. Above all else, I study motion, and in particular how our starships move."`
				goto main
			label taely
			`	When you land on Viminal you spot Taely waiting for you. "Greetings, Captain <first>. It is good to see you again."`
			label main
			`	She makes a gesture you are starting to recognize as a sign of curiosity. "So, Plume sent us some very interesting data, along with a message to expect you with a sample?" You hand her the container with the slime sample and she holds it up to the light. "Interesting looking substance," she chants. "Let's go back to my workshop."`
			choice
				`	"OK, let's go."`
					goto follow

> 				`	"Could I catch up with  you in a minute?"`

Reviewer 2: Double space here (`with  you`)
OK -> Okay

Author: fixed.

event "remnant: penguin"
	shipyard "Remnant Gascraft"
		"Penguin"

mission "Remnant: Face to Maw 2B"
	name "Take the tentacle to Chilia's engineers on Caelian"
	description "The tentacle has properties that may be put to use by the engineers on <planet>."
	to offer
		has "Remnant: Face to Maw 1B: done"
	source "Aventine"
	destination "Caelian"
	cargo "tentacle" 1
	on offer
		conversation
			`When the technician arrives in the spaceport he immediately strides over to you. "I checked the records, and my memory was correct. One of our research teams centuries ago discovered that these tentacles could be used as a devastating weapon in close quarters combat."`

> 			`	He pulls out an image of a strange looking rifle and continues to chant, "The personal weapons lab under Prefect Chilia has a few of the old weapon frames. If you go visit it on Aventine, they could turn this tentacle into a potent weapon for you."`

Reviewer 7: ```suggestion
			`	He pulls out an image of a strange-looking rifle and continues to chant, "The personal weapons lab under Prefect Chilia has a few of the old weapon frames. If you go visit it on Aventine, they could turn this tentacle into a potent weapon for you."`
```
`strange-looking`

			choice
				`	"That sounds good."`
					accept
				`	"I would rather not."`
			`	The technician looks disappointed. "Oh well, enjoy your trophy."`
			decline
	on complete

> 		log "People" "Chilia" `As an military prefect, Chilia is responsible for the defence of Remnant space. He can usually be found on the front lines of combat unless required to work on tactics for the Remnant as a whole.`

Reviewer 7: ```suggestion
		log "People" "Chilia" `As a military prefect, Chilia is responsible for the defence of Remnant space. He can usually be found on the front lines of combat unless required to work on tactics for the Remnant as a whole.`
```

> 		log "People" "Chilia" `As an military prefect, Chilia is responsible for the defence of Remnant space. He can usually be found on the front lines of combat unless required to work on tactics for the Remnant as a whole.`

Reviewer 7: ```suggestion
		log "People" "Chilia" `As an military prefect, Chilia is responsible for the defense of Remnant space. He can usually be found on the front lines of combat unless required to work on tactics for the Remnant as a whole.`
```
double checked, american english prefers `defense`

Author: Fixed. Thanks!

		payment 250000
		"remnant chilia" ++
		outfit "Void Rifle"
		conversation
			branch chilia
				"remnant chilia" > 1

> 			`You follow the directions provided to you and eventually find yourself at a Remnant weapons lab. You enter and are met by security, who ask you to wait. A few minutes later a fit young man emerges from another door. "Greetings, Captain <first>," he sings. "I am Prefect Chilia. I have the nominal responsibility for the overall defense of Remnant space and military operations." He smiles depreciatingly. "I say 'nominal' as virtually all our activities are militaristic in some way, but usually fall under the purview of those who better understand them."`

Reviewer 7: ```suggestion
			`You follow the directions provided to you and eventually find yourself at a Remnant weapons lab. You enter and are met by security, who asks you to wait. A few minutes later a fit young man emerges from another door. "Greetings, Captain <first>," he sings. "I am Prefect Chilia. I have the nominal responsibility for the overall defense of Remnant space and military operations." He smiles depreciatingly. "I say 'nominal' as virtually all our activities are militaristic in some way, but usually fall under the purview of those who better understand them."`
```
`You [...] are met by security, who asks you to wait`

this one's debatable, if you want to be on the safe side, rewrite it to be clearly plural or singular

Reviewer 7: cc @Zireael07

				goto main
			label chilia

> 			`You follow the directions provided to you and eventually find yourself at a Remnant weapons lab. You enter and are met by security, who ask you to wait. A few minutes later a fit young man emerges from another door. "Greetings, captain <first>," he sings. "We meet once again." You immediately recognize the voice as being the man who convinced you to provide the blood sample and get to know the Remnant. "We didn't get a proper introduction last time, but you have probably figured out that I have the nominal responsibility for the overall defense of Remnant space and military operations." He smiles depreciatingly, "I say 'nominal' as virtually all our activities are military in some way, but usually fall under the purview of those who better understand them."`

Reviewer 7: same

Author: OK, both instances have been set to `asks.`

			label main

> 			`	He leads you to a room that looks like an armory where a team of technicians are putting the finishing touches on a rifle stock. "The request from the technician on your behalf caught my attention, as we haven't had the key resources for making void rifles in centuries. Fortunately, we haven't really needed them since we resolved the threats that originally plagued our settlements.`

Reviewer 7: ```suggestion
			`	He leads you to a room that looks like an armory where a team of technicians is putting the finishing touches on a rifle stock. "The request from the technician on your behalf caught my attention, as we haven't had the key resources for making void rifles in centuries. Fortunately, we haven't really needed them since we resolved the threats that originally plagued our settlements.`
```
